--- a/net/minecraft/entity/ai/EntityAIPlay.java
+++ b/net/minecraft/entity/ai/EntityAIPlay.java
@@ -24,7 +24,7 @@
         } else if (this.field_75262_a.func_70681_au().nextInt(400) != 0) {
             return false;
         } else {
-            List list = this.field_75262_a.field_70170_p.func_72872_a(EntityVillager.class, this.field_75262_a.field_70121_D.func_72314_b(6.0, 3.0, 6.0));
+            List<EntityVillager> list = this.field_75262_a.field_70170_p.func_72872_a(EntityVillager.class, this.field_75262_a.field_70121_D.func_72314_b(6.0, 3.0, 6.0));
             double d0 = Double.MAX_VALUE;
 
             for (EntityVillager entityvillager : list) {
