--- a/net/minecraft/command/CommandHandler.java
+++ b/net/minecraft/command/CommandHandler.java
@@ -96,7 +96,7 @@
     }
 
     public ICommand func_71560_a(ICommand p_71560_1_) {
-        List list = p_71560_1_.func_71514_a();
+        List<String> list = p_71560_1_.func_71514_a();
         this.field_71562_a.put(p_71560_1_.func_71517_b(), p_71560_1_);
         this.field_71561_b.add(p_71560_1_);
         if (list != null) {
