--- a/net/minecraft/network/play/client/C01PacketChatMessage.java
+++ b/net/minecraft/network/play/client/C01PacketChatMessage.java
@@ -5,7 +5,7 @@
 import net.minecraft.network.PacketBuffer;
 import net.minecraft.network.play.INetHandlerPlayServer;
 
-public class C01PacketChatMessage extends Packet {
+public class C01PacketChatMessage extends Packet<INetHandlerPlayServer> {
     private String field_149440_a;
 
     public C01PacketChatMessage() {
