--- a/net/minecraft/network/play/client/C03PacketPlayer.java
+++ b/net/minecraft/network/play/client/C03PacketPlayer.java
@@ -7,7 +7,7 @@
 import net.neoforged.api.distmarker.Dist;
 import net.neoforged.api.distmarker.OnlyIn;
 
-public class C03PacketPlayer extends Packet {
+public class C03PacketPlayer extends Packet<INetHandlerPlayServer> {
     protected double field_149479_a;
     protected double field_149477_b;
     protected double field_149478_c;
