--- a/net/minecraft/network/play/client/C07PacketPlayerDigging.java
+++ b/net/minecraft/network/play/client/C07PacketPlayerDigging.java
@@ -7,7 +7,7 @@
 import net.neoforged.api.distmarker.Dist;
 import net.neoforged.api.distmarker.OnlyIn;
 
-public class C07PacketPlayerDigging extends Packet {
+public class C07PacketPlayerDigging extends Packet<INetHandlerPlayServer> {
     private int field_149511_a;
     private int field_149509_b;
     private int field_149510_c;
