--- a/net/minecraft/network/play/client/C08PacketPlayerBlockPlacement.java
+++ b/net/minecraft/network/play/client/C08PacketPlayerBlockPlacement.java
@@ -8,7 +8,7 @@
 import net.neoforged.api.distmarker.Dist;
 import net.neoforged.api.distmarker.OnlyIn;
 
-public class C08PacketPlayerBlockPlacement extends Packet {
+public class C08PacketPlayerBlockPlacement extends Packet<INetHandlerPlayServer> {
     private int field_149583_a;
     private int field_149581_b;
     private int field_149582_c;
