--- a/net/minecraft/network/play/client/C09PacketHeldItemChange.java
+++ b/net/minecraft/network/play/client/C09PacketHeldItemChange.java
@@ -7,7 +7,7 @@
 import net.neoforged.api.distmarker.Dist;
 import net.neoforged.api.distmarker.OnlyIn;
 
-public class C09PacketHeldItemChange extends Packet {
+public class C09PacketHeldItemChange extends Packet<INetHandlerPlayServer> {
     private int field_149615_a;
 
     public C09PacketHeldItemChange() {
