--- a/net/minecraft/network/play/client/C0BPacketEntityAction.java
+++ b/net/minecraft/network/play/client/C0BPacketEntityAction.java
@@ -8,7 +8,7 @@
 import net.neoforged.api.distmarker.Dist;
 import net.neoforged.api.distmarker.OnlyIn;
 
-public class C0BPacketEntityAction extends Packet {
+public class C0BPacketEntityAction extends Packet<INetHandlerPlayServer> {
     private int field_149517_a;
     private int field_149515_b;
     private int field_149516_c;
