--- a/net/minecraft/network/play/client/C0CPacketInput.java
+++ b/net/minecraft/network/play/client/C0CPacketInput.java
@@ -7,7 +7,7 @@
 import net.neoforged.api.distmarker.Dist;
 import net.neoforged.api.distmarker.OnlyIn;
 
-public class C0CPacketInput extends Packet {
+public class C0CPacketInput extends Packet<INetHandlerPlayServer> {
     private float field_149624_a;
     private float field_149622_b;
     private boolean field_149623_c;
