--- a/net/minecraft/network/play/client/C0DPacketCloseWindow.java
+++ b/net/minecraft/network/play/client/C0DPacketCloseWindow.java
@@ -7,7 +7,7 @@
 import net.neoforged.api.distmarker.Dist;
 import net.neoforged.api.distmarker.OnlyIn;
 
-public class C0DPacketCloseWindow extends Packet {
+public class C0DPacketCloseWindow extends Packet<INetHandlerPlayServer> {
     private int field_149556_a;
 
     public C0DPacketCloseWindow() {
