--- a/net/minecraft/network/play/client/C13PacketPlayerAbilities.java
+++ b/net/minecraft/network/play/client/C13PacketPlayerAbilities.java
@@ -6,7 +6,7 @@
 import net.minecraft.network.PacketBuffer;
 import net.minecraft.network.play.INetHandlerPlayServer;
 
-public class C13PacketPlayerAbilities extends Packet {
+public class C13PacketPlayerAbilities extends Packet<INetHandlerPlayServer> {
     private boolean field_149500_a;
     private boolean field_149498_b;
     private boolean field_149499_c;
