--- a/net/minecraft/network/play/client/C16PacketClientStatus.java
+++ b/net/minecraft/network/play/client/C16PacketClientStatus.java
@@ -5,7 +5,7 @@
 import net.minecraft.network.PacketBuffer;
 import net.minecraft.network.play.INetHandlerPlayServer;
 
-public class C16PacketClientStatus extends Packet {
+public class C16PacketClientStatus extends Packet<INetHandlerPlayServer> {
     private C16PacketClientStatus.EnumState field_149437_a;
 
     public C16PacketClientStatus() {
