--- a/net/minecraft/client/audio/SoundManager.java
+++ b/net/minecraft/client/audio/SoundManager.java
@@ -41,11 +41,11 @@
     private SoundManager.SoundSystemStarterThread field_148620_e;
     private boolean field_148617_f;
     private int field_148618_g = 0;
-    private final Map field_148629_h = HashBiMap.create();
+    private final Map<String, ISound> field_148629_h = HashBiMap.create();
     private final Map field_148630_i = ((BiMap)this.field_148629_h).inverse();
     private Map field_148627_j = Maps.newHashMap();
-    private final Multimap field_148628_k = HashMultimap.create();
-    private final List field_148625_l = Lists.newArrayList();
+    private final Multimap<SoundCategory, String> field_148628_k = HashMultimap.create();
+    private final List<ITickableSound> field_148625_l = Lists.newArrayList();
     private final Map field_148626_m = Maps.newHashMap();
     private final Map field_148624_n = Maps.newHashMap();
 
