--- a/net/minecraft/client/renderer/entity/Render.java
+++ b/net/minecraft/client/renderer/entity/Render.java
@@ -18,7 +18,7 @@
 import net.minecraft.world.World;
 import org.lwjgl.opengl.GL11;
 
-public abstract class Render {
+public abstract class Render<T extends Entity> {
     private static final ResourceLocation field_110778_a = new ResourceLocation("textures/misc/shadow.png");
     protected RenderManager field_76990_c;
     protected RenderBlocks field_147909_c = new RenderBlocks();
@@ -26,15 +26,15 @@
     protected float field_76987_f = 1.0F;
     private boolean field_147908_f = false;
 
-    public abstract void func_76986_a(Entity var1, double var2, double var4, double var6, float var8, float var9);
+    public abstract void func_76986_a(T var1, double var2, double var4, double var6, float var8, float var9);
 
-    protected abstract ResourceLocation func_110775_a(Entity var1);
+    protected abstract ResourceLocation func_110775_a(T var1);
 
     public boolean func_147905_a() {
         return this.field_147908_f;
     }
 
-    protected void func_110777_b(Entity p_110777_1_) {
+    protected void func_110777_b(T p_110777_1_) {
         this.func_110776_a(this.func_110775_a(p_110777_1_));
     }
 
