--- a/net/minecraft/client/renderer/entity/RenderSlime.java
+++ b/net/minecraft/client/renderer/entity/RenderSlime.java
@@ -5,7 +5,7 @@
 import net.minecraft.util.ResourceLocation;
 import org.lwjgl.opengl.GL11;
 
-public class RenderSlime extends RenderLiving {
+public class RenderSlime extends RenderLiving<EntitySlime> {
     private static final ResourceLocation field_110897_a = new ResourceLocation("textures/entity/slime/slime.png");
     private ModelBase field_77092_a;
 
