--- a/net/minecraft/client/renderer/entity/RenderSpider.java
+++ b/net/minecraft/client/renderer/entity/RenderSpider.java
@@ -6,7 +6,7 @@
 import net.minecraft.util.ResourceLocation;
 import org.lwjgl.opengl.GL11;
 
-public class RenderSpider extends RenderLiving {
+public class RenderSpider extends RenderLiving<EntitySpider> {
     private static final ResourceLocation field_110891_a = new ResourceLocation("textures/entity/spider_eyes.png");
     private static final ResourceLocation field_110890_f = new ResourceLocation("textures/entity/spider/spider.png");
 
