--- a/net/minecraft/client/renderer/entity/RenderCow.java
+++ b/net/minecraft/client/renderer/entity/RenderCow.java
@@ -4,7 +4,7 @@
 import net.minecraft.entity.passive.EntityCow;
 import net.minecraft.util.ResourceLocation;
 
-public class RenderCow extends RenderLiving {
+public class RenderCow extends RenderLiving<EntityCow> {
     private static final ResourceLocation field_110833_a = new ResourceLocation("textures/entity/cow/cow.png");
 
     public RenderCow(ModelBase p_i1253_1_, float p_i1253_2_) {
