--- a/net/minecraft/client/renderer/entity/RenderSilverfish.java
+++ b/net/minecraft/client/renderer/entity/RenderSilverfish.java
@@ -5,7 +5,7 @@
 import net.minecraft.entity.monster.EntitySilverfish;
 import net.minecraft.util.ResourceLocation;
 
-public class RenderSilverfish extends RenderLiving {
+public class RenderSilverfish extends RenderLiving<EntitySilverfish> {
     private static final ResourceLocation field_110882_a = new ResourceLocation("textures/entity/silverfish.png");
 
     public RenderSilverfish() {
@@ -17,7 +17,7 @@
     }
 
     public void func_76986_a(EntitySilverfish p_76986_1_, double p_76986_2_, double p_76986_4_, double p_76986_6_, float p_76986_8_, float p_76986_9_) {
-        super.func_76986_a((EntityLiving)p_76986_1_, p_76986_2_, p_76986_4_, p_76986_6_, p_76986_8_, p_76986_9_);
+        super.func_76986_a(p_76986_1_, p_76986_2_, p_76986_4_, p_76986_6_, p_76986_8_, p_76986_9_);
     }
 
     protected ResourceLocation func_110775_a(EntitySilverfish p_110775_1_) {
