--- a/net/minecraft/client/renderer/entity/RenderGiantZombie.java
+++ b/net/minecraft/client/renderer/entity/RenderGiantZombie.java
@@ -5,7 +5,7 @@
 import net.minecraft.util.ResourceLocation;
 import org.lwjgl.opengl.GL11;
 
-public class RenderGiantZombie extends RenderLiving {
+public class RenderGiantZombie extends RenderLiving<EntityGiantZombie> {
     private static final ResourceLocation field_110871_a = new ResourceLocation("textures/entity/zombie/zombie.png");
     private float field_77073_a;
 
