--- a/net/minecraft/client/renderer/entity/RenderWither.java
+++ b/net/minecraft/client/renderer/entity/RenderWither.java
@@ -8,7 +8,7 @@
 import net.minecraft.util.ResourceLocation;
 import org.lwjgl.opengl.GL11;
 
-public class RenderWither extends RenderLiving {
+public class RenderWither extends RenderLiving<EntityWither> {
     private static final ResourceLocation field_110913_a = new ResourceLocation("textures/entity/wither/wither_invulnerable.png");
     private static final ResourceLocation field_110912_f = new ResourceLocation("textures/entity/wither/wither.png");
     private int field_82419_a = ((ModelWither)this.field_77045_g).func_82903_a();
@@ -25,7 +25,7 @@
             this.field_77045_g = new ModelWither();
         }
 
-        super.func_76986_a((EntityLiving)p_76986_1_, p_76986_2_, p_76986_4_, p_76986_6_, p_76986_8_, p_76986_9_);
+        super.func_76986_a(p_76986_1_, p_76986_2_, p_76986_4_, p_76986_6_, p_76986_8_, p_76986_9_);
     }
 
     protected ResourceLocation func_110775_a(EntityWither p_110775_1_) {
