--- a/net/minecraft/client/renderer/entity/RenderPig.java
+++ b/net/minecraft/client/renderer/entity/RenderPig.java
@@ -4,7 +4,7 @@
 import net.minecraft.entity.passive.EntityPig;
 import net.minecraft.util.ResourceLocation;
 
-public class RenderPig extends RenderLiving {
+public class RenderPig extends RenderLiving<EntityPig> {
     private static final ResourceLocation field_110888_a = new ResourceLocation("textures/entity/pig/pig_saddle.png");
     private static final ResourceLocation field_110887_f = new ResourceLocation("textures/entity/pig/pig.png");
 
