--- a/net/minecraft/client/renderer/entity/RenderXPOrb.java
+++ b/net/minecraft/client/renderer/entity/RenderXPOrb.java
@@ -7,7 +7,7 @@
 import net.minecraft.util.ResourceLocation;
 import org.lwjgl.opengl.GL11;
 
-public class RenderXPOrb extends Render {
+public class RenderXPOrb extends Render<EntityXPOrb> {
     private static final ResourceLocation field_110785_a = new ResourceLocation("textures/entity/experience_orb.png");
 
     public RenderXPOrb() {
