--- a/net/minecraft/client/renderer/entity/RenderWolf.java
+++ b/net/minecraft/client/renderer/entity/RenderWolf.java
@@ -6,7 +6,7 @@
 import net.minecraft.util.ResourceLocation;
 import org.lwjgl.opengl.GL11;
 
-public class RenderWolf extends RenderLiving {
+public class RenderWolf extends RenderLiving<EntityWolf> {
     private static final ResourceLocation field_110917_a = new ResourceLocation("textures/entity/wolf/wolf.png");
     private static final ResourceLocation field_110915_f = new ResourceLocation("textures/entity/wolf/wolf_tame.png");
     private static final ResourceLocation field_110916_g = new ResourceLocation("textures/entity/wolf/wolf_angry.png");
