--- a/net/minecraft/client/renderer/entity/RenderFireball.java
+++ b/net/minecraft/client/renderer/entity/RenderFireball.java
@@ -8,7 +8,7 @@
 import net.minecraft.util.ResourceLocation;
 import org.lwjgl.opengl.GL11;
 
-public class RenderFireball extends Render {
+public class RenderFireball extends Render<EntityFireball> {
     private float field_77002_a;
 
     public RenderFireball(float p_i1254_1_) {
