--- a/net/minecraft/client/renderer/entity/RenderMooshroom.java
+++ b/net/minecraft/client/renderer/entity/RenderMooshroom.java
@@ -9,7 +9,7 @@
 import net.minecraft.util.ResourceLocation;
 import org.lwjgl.opengl.GL11;
 
-public class RenderMooshroom extends RenderLiving {
+public class RenderMooshroom extends RenderLiving<EntityMooshroom> {
     private static final ResourceLocation field_110880_a = new ResourceLocation("textures/entity/cow/mooshroom.png");
 
     public RenderMooshroom(ModelBase p_i1263_1_, float p_i1263_2_) {
@@ -17,7 +17,7 @@
     }
 
     public void func_76986_a(EntityMooshroom p_76986_1_, double p_76986_2_, double p_76986_4_, double p_76986_6_, float p_76986_8_, float p_76986_9_) {
-        super.func_76986_a((EntityLiving)p_76986_1_, p_76986_2_, p_76986_4_, p_76986_6_, p_76986_8_, p_76986_9_);
+        super.func_76986_a(p_76986_1_, p_76986_2_, p_76986_4_, p_76986_6_, p_76986_8_, p_76986_9_);
     }
 
     protected ResourceLocation func_110775_a(EntityMooshroom p_110775_1_) {
