--- a/net/minecraft/client/renderer/entity/RenderFallingBlock.java
+++ b/net/minecraft/client/renderer/entity/RenderFallingBlock.java
@@ -12,7 +12,7 @@
 import net.minecraft.world.World;
 import org.lwjgl.opengl.GL11;
 
-public class RenderFallingBlock extends Render {
+public class RenderFallingBlock extends Render<EntityFallingBlock> {
     private final RenderBlocks field_147920_a = new RenderBlocks();
 
     public RenderFallingBlock() {
