--- a/net/minecraft/client/renderer/entity/RenderMinecart.java
+++ b/net/minecraft/client/renderer/entity/RenderMinecart.java
@@ -11,7 +11,7 @@
 import net.minecraft.util.Vec3;
 import org.lwjgl.opengl.GL11;
 
-public class RenderMinecart extends Render {
+public class RenderMinecart extends Render<EntityMinecart> {
     private static final ResourceLocation field_110804_g = new ResourceLocation("textures/entity/minecart.png");
     protected ModelBase field_77013_a = new ModelMinecart();
     protected final RenderBlocks field_94145_f;
