--- a/net/minecraft/client/renderer/entity/RenderItem.java
+++ b/net/minecraft/client/renderer/entity/RenderItem.java
@@ -24,7 +24,7 @@
 import net.minecraft.util.ResourceLocation;
 import org.lwjgl.opengl.GL11;
 
-public class RenderItem extends Render {
+public class RenderItem extends Render<EntityItem> {
     private static final ResourceLocation field_110798_h = new ResourceLocation("textures/misc/enchanted_item_glint.png");
     private RenderBlocks field_147913_i = new RenderBlocks();
     private Random field_77025_h = new Random();
