--- a/net/minecraft/client/util/QuadComparator.java
+++ b/net/minecraft/client/util/QuadComparator.java
@@ -5,7 +5,7 @@
 import net.neoforged.api.distmarker.OnlyIn;
 
 @OnlyIn(Dist.CLIENT)
-public class QuadComparator implements Comparator {
+public class QuadComparator implements Comparator<Integer> {
     private float field_147630_a;
     private float field_147628_b;
     private float field_147629_c;
