--- a/net/minecraft/client/settings/KeyBinding.java
+++ b/net/minecraft/client/settings/KeyBinding.java
@@ -10,8 +10,8 @@
 import net.neoforged.api.distmarker.OnlyIn;
 
 @OnlyIn(Dist.CLIENT)
-public class KeyBinding implements Comparable {
-    private static final List field_74516_a = new ArrayList();
+public class KeyBinding implements Comparable<KeyBinding> {
+    private static final List<KeyBinding> field_74516_a = new ArrayList();
     private static final IntHashMap field_74514_b = new IntHashMap();
     private static final Set field_151473_c = new HashSet();
     private final String field_74515_c;
