--- a/net/minecraft/network/play/client/C00PacketKeepAlive.java
+++ b/net/minecraft/network/play/client/C00PacketKeepAlive.java
@@ -5,7 +5,7 @@
 import net.minecraft.network.PacketBuffer;
 import net.minecraft.network.play.INetHandlerPlayServer;
 
-public class C00PacketKeepAlive extends Packet {
+public class C00PacketKeepAlive extends Packet<INetHandlerPlayServer> {
     private int field_149461_a;
 
     public C00PacketKeepAlive() {
