--- a/net/minecraft/network/play/client/C0APacketAnimation.java
+++ b/net/minecraft/network/play/client/C0APacketAnimation.java
@@ -6,7 +6,7 @@
 import net.minecraft.network.PacketBuffer;
 import net.minecraft.network.play.INetHandlerPlayServer;
 
-public class C0APacketAnimation extends Packet {
+public class C0APacketAnimation extends Packet<INetHandlerPlayServer> {
     private int field_149424_a;
     private int field_149423_b;
 
