--- a/net/minecraft/network/play/client/C0FPacketConfirmTransaction.java
+++ b/net/minecraft/network/play/client/C0FPacketConfirmTransaction.java
@@ -5,7 +5,7 @@
 import net.minecraft.network.PacketBuffer;
 import net.minecraft.network.play.INetHandlerPlayServer;
 
-public class C0FPacketConfirmTransaction extends Packet {
+public class C0FPacketConfirmTransaction extends Packet<INetHandlerPlayServer> {
     private int field_149536_a;
     private short field_149534_b;
     private boolean field_149535_c;
