--- a/net/minecraft/network/play/client/C10PacketCreativeInventoryAction.java
+++ b/net/minecraft/network/play/client/C10PacketCreativeInventoryAction.java
@@ -6,7 +6,7 @@
 import net.minecraft.network.PacketBuffer;
 import net.minecraft.network.play.INetHandlerPlayServer;
 
-public class C10PacketCreativeInventoryAction extends Packet {
+public class C10PacketCreativeInventoryAction extends Packet<INetHandlerPlayServer> {
     private int field_149629_a;
     private ItemStack field_149628_b;
 
