--- a/net/minecraft/network/play/client/C14PacketTabComplete.java
+++ b/net/minecraft/network/play/client/C14PacketTabComplete.java
@@ -6,7 +6,7 @@
 import net.minecraft.network.play.INetHandlerPlayServer;
 import org.apache.commons.lang3.StringUtils;
 
-public class C14PacketTabComplete extends Packet {
+public class C14PacketTabComplete extends Packet<INetHandlerPlayServer> {
     private String field_149420_a;
 
     public C14PacketTabComplete() {
