--- a/net/minecraft/network/play/client/C15PacketClientSettings.java
+++ b/net/minecraft/network/play/client/C15PacketClientSettings.java
@@ -7,7 +7,7 @@
 import net.minecraft.network.play.INetHandlerPlayServer;
 import net.minecraft.world.EnumDifficulty;
 
-public class C15PacketClientSettings extends Packet {
+public class C15PacketClientSettings extends Packet<INetHandlerPlayServer> {
     private String field_149530_a;
     private int field_149528_b;
     private EntityPlayer.EnumChatVisibility field_149529_c;
