--- a/net/minecraft/util/ChatStyle.java
+++ b/net/minecraft/util/ChatStyle.java
@@ -294,7 +294,7 @@
         return chatstyle;
     }
 
-    public static class Serializer implements JsonDeserializer, JsonSerializer {
+    public static class Serializer implements JsonDeserializer<ChatStyle>, JsonSerializer<ChatStyle> {
         public ChatStyle deserialize(JsonElement p_deserialize_1_, Type p_deserialize_2_, JsonDeserializationContext p_deserialize_3_) {
             if (p_deserialize_1_.isJsonObject()) {
                 ChatStyle chatstyle = new ChatStyle();
