--- a/net/minecraft/util/HttpUtil.java
+++ b/net/minecraft/util/HttpUtil.java
@@ -19,7 +19,7 @@
     private static final AtomicInteger field_151228_a = new AtomicInteger(0);
     private static final Logger field_151227_b = LogManager.getLogger();
 
-    public static String func_76179_a(Map p_76179_0_) {
+    public static String func_76179_a(Map<String, Object> p_76179_0_) {
         StringBuilder stringbuilder = new StringBuilder();
 
         for (Entry entry : p_76179_0_.entrySet()) {
