--- a/net/minecraft/util/Tuple.java
+++ b/net/minecraft/util/Tuple.java
@@ -1,19 +1,19 @@
 package net.minecraft.util;
 
-public class Tuple {
-    private Object field_76343_a;
-    private Object field_76342_b;
+public class Tuple<A, B> {
+    private A field_76343_a;
+    private B field_76342_b;
 
-    public Tuple(Object p_i1555_1_, Object p_i1555_2_) {
+    public Tuple(A p_i1555_1_, B p_i1555_2_) {
         this.field_76343_a = p_i1555_1_;
         this.field_76342_b = p_i1555_2_;
     }
 
-    public Object func_76341_a() {
+    public A func_76341_a() {
         return this.field_76343_a;
     }
 
-    public Object func_76340_b() {
+    public B func_76340_b() {
         return this.field_76342_b;
     }
 }
