--- a/net/minecraft/network/NettyEncryptingEncoder.java
+++ b/net/minecraft/network/NettyEncryptingEncoder.java
@@ -6,7 +6,7 @@
 import javax.crypto.Cipher;
 import javax.crypto.ShortBufferException;
 
-public class NettyEncryptingEncoder extends MessageToByteEncoder {
+public class NettyEncryptingEncoder extends MessageToByteEncoder<ByteBuf> {
     private final NettyEncryptionTranslator field_150750_a;
 
     public NettyEncryptingEncoder(Cipher p_i45142_1_) {
