--- a/net/minecraft/block/BlockPressurePlate.java
+++ b/net/minecraft/block/BlockPressurePlate.java
@@ -27,7 +27,7 @@
 
     @Override
     protected int func_150065_e(World p_150065_1_, int p_150065_2_, int p_150065_3_, int p_150065_4_) {
-        List list = null;
+        List<? extends Entity> list = null;
         if (this.field_150069_a == BlockPressurePlate.Sensitivity.everything) {
             list = p_150065_1_.func_72839_b(null, this.func_150061_a(p_150065_2_, p_150065_3_, p_150065_4_));
         }
