--- a/net/minecraft/network/Packet.java
+++ b/net/minecraft/network/Packet.java
@@ -6,7 +6,7 @@
 import org.apache.logging.log4j.LogManager;
 import org.apache.logging.log4j.Logger;
 
-public abstract class Packet {
+public abstract class Packet<T extends INetHandler> {
     private static final Logger field_148841_a = LogManager.getLogger();
 
     public static Packet func_148839_a(BiMap p_148839_0_, int p_148839_1_) {
@@ -39,7 +39,7 @@
 
     public abstract void func_148840_b(PacketBuffer var1) throws IOException;
 
-    public abstract void func_148833_a(INetHandler var1);
+    public abstract void func_148833_a(T var1);
 
     public boolean func_148836_a() {
         return false;
