--- a/net/minecraft/world/gen/structure/MapGenStructure.java
+++ b/net/minecraft/world/gen/structure/MapGenStructure.java
@@ -137,7 +137,7 @@
         if (chunkposition != null) {
             return chunkposition;
         } else {
-            List list = this.func_75052_o_();
+            List<ChunkPosition> list = this.func_75052_o_();
             if (list != null) {
                 ChunkPosition chunkposition2 = null;
 
