--- a/net/minecraft/world/gen/structure/MapGenScatteredFeature.java
+++ b/net/minecraft/world/gen/structure/MapGenScatteredFeature.java
@@ -12,7 +12,7 @@
 import net.minecraft.world.biome.BiomeGenBase;
 
 public class MapGenScatteredFeature extends MapGenStructure {
-    private static List field_75061_e = Arrays.asList(
+    private static List<BiomeGenBase> field_75061_e = Arrays.asList(
         BiomeGenBase.field_76769_d, BiomeGenBase.field_76786_s, BiomeGenBase.field_76782_w, BiomeGenBase.field_76792_x, BiomeGenBase.field_76780_h
     );
     private List field_82668_f = new ArrayList();
@@ -23,7 +23,7 @@
         this.field_82668_f.add(new BiomeGenBase.SpawnListEntry(EntityWitch.class, 1, 1, 1));
     }
 
-    public MapGenScatteredFeature(Map p_i2061_1_) {
+    public MapGenScatteredFeature(Map<String, String> p_i2061_1_) {
         this();
 
         for (Entry entry : p_i2061_1_.entrySet()) {
