--- a/net/minecraft/world/gen/structure/StructureStart.java
+++ b/net/minecraft/world/gen/structure/StructureStart.java
@@ -8,7 +8,7 @@
 import net.minecraft.world.World;
 
 public abstract class StructureStart {
-    protected LinkedList field_75075_a = new LinkedList();
+    protected LinkedList<StructureComponent> field_75075_a = new LinkedList();
     protected StructureBoundingBox field_75074_b;
     private int field_143024_c;
     private int field_143023_d;
@@ -25,7 +25,7 @@
         return this.field_75074_b;
     }
 
-    public LinkedList func_75073_b() {
+    public LinkedList<StructureComponent> func_75073_b() {
         return this.field_75075_a;
     }
 
