--- a/net/minecraft/world/gen/structure/StructureVillagePieces.java
+++ b/net/minecraft/world/gen/structure/StructureVillagePieces.java
@@ -80,7 +80,7 @@
         return arraylist;
     }
 
-    private static int func_75079_a(List p_75079_0_) {
+    private static int func_75079_a(List<StructureVillagePieces.PieceWeight> p_75079_0_) {
         boolean flag = false;
         int i = 0;
 
@@ -1448,9 +1448,9 @@
         public boolean field_74927_b;
         public int field_74928_c;
         public StructureVillagePieces.PieceWeight field_74926_d;
-        public List field_74931_h;
-        public List field_74932_i = new ArrayList();
-        public List field_74930_j = new ArrayList();
+        public List<StructureVillagePieces.PieceWeight> field_74931_h;
+        public List<StructureVillagePieces.Village> field_74932_i = new ArrayList();
+        public List<StructureVillagePieces.Path> field_74930_j = new ArrayList();
 
         public Start() {
         }
