--- a/net/minecraft/server/MinecraftServer.java
+++ b/net/minecraft/server/MinecraftServer.java
@@ -764,12 +764,12 @@
         return p_71230_1_;
     }
 
-    public List func_71248_a(ICommandSender p_71248_1_, String p_71248_2_) {
+    public List<String> func_71248_a(ICommandSender p_71248_1_, String p_71248_2_) {
         ArrayList arraylist = new ArrayList();
         if (p_71248_2_.startsWith("/")) {
             p_71248_2_ = p_71248_2_.substring(1);
             boolean flag = !p_71248_2_.contains(" ");
-            List list = this.field_71321_q.func_71558_b(p_71248_1_, p_71248_2_);
+            List<String> list = this.field_71321_q.func_71558_b(p_71248_1_, p_71248_2_);
             if (list != null) {
                 for (String s2 : list) {
                     if (flag) {
