--- a/net/minecraft/util/IChatComponent.java
+++ b/net/minecraft/util/IChatComponent.java
@@ -15,7 +15,7 @@
 import java.util.List;
 import java.util.Map.Entry;
 
-public interface IChatComponent extends Iterable {
+public interface IChatComponent extends Iterable<IChatComponent> {
     IChatComponent func_150255_a(ChatStyle var1);
 
     ChatStyle func_150256_b();
@@ -30,11 +30,11 @@
 
     String func_150254_d();
 
-    List func_150253_a();
+    List<IChatComponent> func_150253_a();
 
     IChatComponent func_150259_f();
 
-    public static class Serializer implements JsonDeserializer, JsonSerializer {
+    public static class Serializer implements JsonDeserializer<IChatComponent>, JsonSerializer<IChatComponent> {
         private static final Gson field_150700_a;
 
         public IChatComponent deserialize(JsonElement p_deserialize_1_, Type p_deserialize_2_, JsonDeserializationContext p_deserialize_3_) {
