--- a/net/minecraft/world/WorldServer.java
+++ b/net/minecraft/world/WorldServer.java
@@ -544,7 +544,7 @@
         return this.field_73059_b;
     }
 
-    public List func_147486_a(int p_147486_1_, int p_147486_2_, int p_147486_3_, int p_147486_4_, int p_147486_5_, int p_147486_6_) {
+    public List<TileEntity> func_147486_a(int p_147486_1_, int p_147486_2_, int p_147486_3_, int p_147486_4_, int p_147486_5_, int p_147486_6_) {
         ArrayList arraylist = new ArrayList();
 
         for (int i = 0; i < this.field_147482_g.size(); i++) {
@@ -896,7 +896,7 @@
         }
     }
 
-    static class ServerBlockEventList extends ArrayList {
+    static class ServerBlockEventList extends ArrayList<BlockEventData> {
         private ServerBlockEventList() {
         }
     }
