--- a/net/minecraft/client/gui/inventory/GuiContainerCreative.java
+++ b/net/minecraft/client/gui/inventory/GuiContainerCreative.java
@@ -471,7 +471,7 @@
             List list = p_146285_1_.func_82840_a(this.field_146297_k.field_71439_g, this.field_146297_k.field_71474_y.field_82882_x);
             CreativeTabs creativetabs = p_146285_1_.func_77973_b().func_77640_w();
             if (creativetabs == null && p_146285_1_.func_77973_b() == Items.field_151134_bR) {
-                Map map = EnchantmentHelper.func_82781_a(p_146285_1_);
+                Map<Integer, Integer> map = EnchantmentHelper.func_82781_a(p_146285_1_);
                 if (map.size() == 1) {
                     Enchantment enchantment = Enchantment.field_77331_b[map.keySet().iterator().next()];
 
