--- a/net/minecraft/client/gui/inventory/GuiContainer.java
+++ b/net/minecraft/client/gui/inventory/GuiContainer.java
@@ -36,7 +36,7 @@
     private ItemStack field_146991_C;
     private Slot field_146985_D;
     private long field_146986_E;
-    protected final Set field_147008_s = new HashSet();
+    protected final Set<Slot> field_147008_s = new HashSet();
     protected boolean field_147007_t;
     private int field_146987_F;
     private int field_146988_G;
