--- a/net/minecraft/client/particle/EffectRenderer.java
+++ b/net/minecraft/client/particle/EffectRenderer.java
@@ -51,7 +51,8 @@
     }
 
     public void func_78868_a() {
-        for (final int i = 0; i < 4; i++) {
+        for (int mutablei = 0; mutablei < 4; mutablei++) {
+            final int i = mutablei;
             for (int j = 0; j < this.field_78876_b[i].size(); j++) {
                 final EntityFX entityfx = this.field_78876_b[i].get(j);
 
@@ -98,7 +99,8 @@
         EntityFX.field_70554_ao = p_78874_1_.field_70137_T + (p_78874_1_.field_70163_u - p_78874_1_.field_70137_T) * p_78874_2_;
         EntityFX.field_70555_ap = p_78874_1_.field_70136_U + (p_78874_1_.field_70161_v - p_78874_1_.field_70136_U) * p_78874_2_;
 
-        for (final int i = 0; i < 3; i++) {
+        for (int mutablei = 0; mutablei < 3; mutablei++) {
+            final int i = mutablei;
             if (!this.field_78876_b[i].isEmpty()) {
                 switch (i) {
                     case 0:
