--- a/net/minecraft/entity/ai/attributes/BaseAttributeMap.java
+++ b/net/minecraft/entity/ai/attributes/BaseAttributeMap.java
@@ -8,8 +8,8 @@
 import net.minecraft.server.management.LowerStringMap;
 
 public abstract class BaseAttributeMap {
-    protected final Map field_111154_a = new HashMap();
-    protected final Map field_111153_b = new LowerStringMap();
+    protected final Map<IAttribute, IAttributeInstance> field_111154_a = new HashMap();
+    protected final Map<String, IAttributeInstance> field_111153_b = new LowerStringMap();
 
     public IAttributeInstance func_111151_a(IAttribute p_111151_1_) {
         return (IAttributeInstance)this.field_111154_a.get(p_111151_1_);
@@ -21,14 +21,14 @@
 
     public abstract IAttributeInstance func_111150_b(IAttribute var1);
 
-    public Collection func_111146_a() {
+    public Collection<IAttributeInstance> func_111146_a() {
         return this.field_111153_b.values();
     }
 
     public void func_111149_a(ModifiableAttributeInstance p_111149_1_) {
     }
 
-    public void func_111148_a(Multimap p_111148_1_) {
+    public void func_111148_a(Multimap<String, AttributeModifier> p_111148_1_) {
         for (Entry entry : p_111148_1_.entries()) {
             IAttributeInstance iattributeinstance = this.func_111152_a((String)entry.getKey());
             if (iattributeinstance != null) {
@@ -37,7 +37,7 @@
         }
     }
 
-    public void func_111147_b(Multimap p_111147_1_) {
+    public void func_111147_b(Multimap<String, AttributeModifier> p_111147_1_) {
         for (Entry entry : p_111147_1_.entries()) {
             IAttributeInstance iattributeinstance = this.func_111152_a((String)entry.getKey());
             if (iattributeinstance != null) {
