--- a/net/minecraft/world/gen/ChunkProviderFlat.java
+++ b/net/minecraft/world/gen/ChunkProviderFlat.java
@@ -28,7 +28,7 @@
     private final Block[] field_82700_c = new Block[256];
     private final byte[] field_82698_d = new byte[256];
     private final FlatGeneratorInfo field_82699_e;
-    private final List field_82696_f = new ArrayList();
+    private final List<MapGenStructure> field_82696_f = new ArrayList();
     private final boolean field_82697_g;
     private final boolean field_82702_h;
     private WorldGenLakes field_82703_i;
