--- a/net/minecraft/network/login/client/C01PacketEncryptionResponse.java
+++ b/net/minecraft/network/login/client/C01PacketEncryptionResponse.java
@@ -8,7 +8,7 @@
 import net.minecraft.network.login.INetHandlerLoginServer;
 import net.minecraft.util.CryptManager;
 
-public class C01PacketEncryptionResponse extends Packet {
+public class C01PacketEncryptionResponse extends Packet<INetHandlerLoginServer> {
     private byte[] field_149302_a = new byte[0];
     private byte[] field_149301_b = new byte[0];
 
