--- a/net/minecraft/inventory/ContainerRepair.java
+++ b/net/minecraft/inventory/ContainerRepair.java
@@ -128,7 +128,7 @@
         } else {
             ItemStack itemstack1 = itemstack.func_77946_l();
             ItemStack itemstack2 = this.field_82853_g.func_70301_a(1);
-            Map map = EnchantmentHelper.func_82781_a(itemstack1);
+            Map<Integer, Integer> map = EnchantmentHelper.func_82781_a(itemstack1);
             boolean flag = false;
             j += itemstack.func_82838_A() + (itemstack2 == null ? 0 : itemstack2.func_82838_A());
             this.field_82856_l = 0;
@@ -174,7 +174,7 @@
                         }
                     }
 
-                    Map map1 = EnchantmentHelper.func_82781_a(itemstack2);
+                    Map<Integer, Integer> map1 = EnchantmentHelper.func_82781_a(itemstack2);
 
                     for (int l3 : map1.keySet()) {
                         Enchantment enchantment = Enchantment.field_77331_b[l3];
