--- a/net/minecraft/network/status/server/S00PacketServerInfo.java
+++ b/net/minecraft/network/status/server/S00PacketServerInfo.java
@@ -11,7 +11,7 @@
 import net.minecraft.util.EnumTypeAdapterFactory;
 import net.minecraft.util.IChatComponent;
 
-public class S00PacketServerInfo extends Packet {
+public class S00PacketServerInfo extends Packet<INetHandlerStatusClient> {
     private static final Gson field_149297_a = new GsonBuilder()
         .registerTypeAdapter(
             ServerStatusResponse.MinecraftProtocolVersionIdentifier.class, new ServerStatusResponse.MinecraftProtocolVersionIdentifier.Serializer()
