--- a/net/minecraft/network/play/server/S02PacketChat.java
+++ b/net/minecraft/network/play/server/S02PacketChat.java
@@ -8,7 +8,7 @@
 import net.neoforged.api.distmarker.Dist;
 import net.neoforged.api.distmarker.OnlyIn;
 
-public class S02PacketChat extends Packet {
+public class S02PacketChat extends Packet<INetHandlerPlayClient> {
     private IChatComponent field_148919_a;
     private boolean field_148918_b = true;
 
