--- a/net/minecraft/network/play/server/S03PacketTimeUpdate.java
+++ b/net/minecraft/network/play/server/S03PacketTimeUpdate.java
@@ -7,7 +7,7 @@
 import net.neoforged.api.distmarker.Dist;
 import net.neoforged.api.distmarker.OnlyIn;
 
-public class S03PacketTimeUpdate extends Packet {
+public class S03PacketTimeUpdate extends Packet<INetHandlerPlayClient> {
     private long field_149369_a;
     private long field_149368_b;
 
