--- a/net/minecraft/network/play/server/S06PacketUpdateHealth.java
+++ b/net/minecraft/network/play/server/S06PacketUpdateHealth.java
@@ -7,7 +7,7 @@
 import net.neoforged.api.distmarker.Dist;
 import net.neoforged.api.distmarker.OnlyIn;
 
-public class S06PacketUpdateHealth extends Packet {
+public class S06PacketUpdateHealth extends Packet<INetHandlerPlayClient> {
     private float field_149336_a;
     private int field_149334_b;
     private float field_149335_c;
