--- a/net/minecraft/network/play/server/S08PacketPlayerPosLook.java
+++ b/net/minecraft/network/play/server/S08PacketPlayerPosLook.java
@@ -7,7 +7,7 @@
 import net.neoforged.api.distmarker.Dist;
 import net.neoforged.api.distmarker.OnlyIn;
 
-public class S08PacketPlayerPosLook extends Packet {
+public class S08PacketPlayerPosLook extends Packet<INetHandlerPlayClient> {
     private double field_148940_a;
     private double field_148938_b;
     private double field_148939_c;
