--- a/net/minecraft/network/play/server/S13PacketDestroyEntities.java
+++ b/net/minecraft/network/play/server/S13PacketDestroyEntities.java
@@ -7,7 +7,7 @@
 import net.neoforged.api.distmarker.Dist;
 import net.neoforged.api.distmarker.OnlyIn;
 
-public class S13PacketDestroyEntities extends Packet {
+public class S13PacketDestroyEntities extends Packet<INetHandlerPlayClient> {
     private int[] field_149100_a;
 
     public S13PacketDestroyEntities() {
