--- a/net/minecraft/network/play/server/S18PacketEntityTeleport.java
+++ b/net/minecraft/network/play/server/S18PacketEntityTeleport.java
@@ -9,7 +9,7 @@
 import net.neoforged.api.distmarker.Dist;
 import net.neoforged.api.distmarker.OnlyIn;
 
-public class S18PacketEntityTeleport extends Packet {
+public class S18PacketEntityTeleport extends Packet<INetHandlerPlayClient> {
     private int field_149458_a;
     private int field_149456_b;
     private int field_149457_c;
