--- a/net/minecraft/network/play/server/S1DPacketEntityEffect.java
+++ b/net/minecraft/network/play/server/S1DPacketEntityEffect.java
@@ -8,7 +8,7 @@
 import net.neoforged.api.distmarker.Dist;
 import net.neoforged.api.distmarker.OnlyIn;
 
-public class S1DPacketEntityEffect extends Packet {
+public class S1DPacketEntityEffect extends Packet<INetHandlerPlayClient> {
     private int field_149434_a;
     private byte field_149432_b;
     private byte field_149433_c;
