--- a/net/minecraft/network/play/server/S1FPacketSetExperience.java
+++ b/net/minecraft/network/play/server/S1FPacketSetExperience.java
@@ -7,7 +7,7 @@
 import net.neoforged.api.distmarker.Dist;
 import net.neoforged.api.distmarker.OnlyIn;
 
-public class S1FPacketSetExperience extends Packet {
+public class S1FPacketSetExperience extends Packet<INetHandlerPlayClient> {
     private float field_149401_a;
     private int field_149399_b;
     private int field_149400_c;
