--- a/net/minecraft/network/play/server/S24PacketBlockAction.java
+++ b/net/minecraft/network/play/server/S24PacketBlockAction.java
@@ -8,7 +8,7 @@
 import net.neoforged.api.distmarker.Dist;
 import net.neoforged.api.distmarker.OnlyIn;
 
-public class S24PacketBlockAction extends Packet {
+public class S24PacketBlockAction extends Packet<INetHandlerPlayClient> {
     private int field_148876_a;
     private int field_148874_b;
     private int field_148875_c;
