--- a/net/minecraft/network/play/server/S25PacketBlockBreakAnim.java
+++ b/net/minecraft/network/play/server/S25PacketBlockBreakAnim.java
@@ -7,7 +7,7 @@
 import net.neoforged.api.distmarker.Dist;
 import net.neoforged.api.distmarker.OnlyIn;
 
-public class S25PacketBlockBreakAnim extends Packet {
+public class S25PacketBlockBreakAnim extends Packet<INetHandlerPlayClient> {
     private int field_148852_a;
     private int field_148850_b;
     private int field_148851_c;
