--- a/net/minecraft/network/play/server/S28PacketEffect.java
+++ b/net/minecraft/network/play/server/S28PacketEffect.java
@@ -7,7 +7,7 @@
 import net.neoforged.api.distmarker.Dist;
 import net.neoforged.api.distmarker.OnlyIn;
 
-public class S28PacketEffect extends Packet {
+public class S28PacketEffect extends Packet<INetHandlerPlayClient> {
     private int field_149251_a;
     private int field_149249_b;
     private int field_149250_c;
