--- a/net/minecraft/network/play/server/S37PacketStatistics.java
+++ b/net/minecraft/network/play/server/S37PacketStatistics.java
@@ -12,8 +12,8 @@
 import net.neoforged.api.distmarker.Dist;
 import net.neoforged.api.distmarker.OnlyIn;
 
-public class S37PacketStatistics extends Packet {
-    private Map field_148976_a;
+public class S37PacketStatistics extends Packet<INetHandlerPlayClient> {
+    private Map<StatBase, Integer> field_148976_a;
 
     public S37PacketStatistics() {
     }
@@ -56,7 +56,7 @@
     }
 
     @OnlyIn(Dist.CLIENT)
-    public Map func_148974_c() {
+    public Map<StatBase, Integer> func_148974_c() {
         return this.field_148976_a;
     }
 }
