--- a/net/minecraft/network/play/server/S39PacketPlayerAbilities.java
+++ b/net/minecraft/network/play/server/S39PacketPlayerAbilities.java
@@ -6,7 +6,7 @@
 import net.minecraft.network.PacketBuffer;
 import net.minecraft.network.play.INetHandlerPlayClient;
 
-public class S39PacketPlayerAbilities extends Packet {
+public class S39PacketPlayerAbilities extends Packet<INetHandlerPlayClient> {
     private boolean field_149119_a;
     private boolean field_149117_b;
     private boolean field_149118_c;
