--- a/net/minecraft/network/play/server/S3BPacketScoreboardObjective.java
+++ b/net/minecraft/network/play/server/S3BPacketScoreboardObjective.java
@@ -8,7 +8,7 @@
 import net.neoforged.api.distmarker.Dist;
 import net.neoforged.api.distmarker.OnlyIn;
 
-public class S3BPacketScoreboardObjective extends Packet {
+public class S3BPacketScoreboardObjective extends Packet<INetHandlerPlayClient> {
     private String field_149343_a;
     private String field_149341_b;
     private int field_149342_c;
