--- a/net/minecraft/network/play/server/S3EPacketTeams.java
+++ b/net/minecraft/network/play/server/S3EPacketTeams.java
@@ -10,12 +10,12 @@
 import net.neoforged.api.distmarker.Dist;
 import net.neoforged.api.distmarker.OnlyIn;
 
-public class S3EPacketTeams extends Packet {
+public class S3EPacketTeams extends Packet<INetHandlerPlayClient> {
     private String field_149320_a = "";
     private String field_149318_b = "";
     private String field_149319_c = "";
     private String field_149316_d = "";
-    private Collection field_149317_e = new ArrayList();
+    private Collection<String> field_149317_e = new ArrayList();
     private int field_149314_f;
     private int field_149315_g;
 
@@ -114,7 +114,7 @@
     }
 
     @OnlyIn(Dist.CLIENT)
-    public Collection func_149310_g() {
+    public Collection<String> func_149310_g() {
         return this.field_149317_e;
     }
 
