--- a/net/minecraft/network/EnumConnectionState.java
+++ b/net/minecraft/network/EnumConnectionState.java
@@ -220,10 +220,10 @@
     };
 
     private static final TIntObjectMap field_150764_e = new TIntObjectHashMap();
-    private static final Map field_150761_f = Maps.newHashMap();
+    private static final Map<Class, EnumConnectionState> field_150761_f = Maps.newHashMap();
     private final int field_150762_g;
-    private final BiMap field_150769_h = HashBiMap.create();
-    private final BiMap field_150770_i = HashBiMap.create();
+    private final BiMap<Integer, Class> field_150769_h = HashBiMap.create();
+    private final BiMap<Integer, Class> field_150770_i = HashBiMap.create();
 
     private EnumConnectionState(int p_i45152_3_) {
         this.field_150762_g = p_i45152_3_;
@@ -279,19 +279,19 @@
         }
     }
 
-    public BiMap func_150753_a() {
+    public BiMap<Integer, Class> func_150753_a() {
         return this.field_150769_h;
     }
 
-    public BiMap func_150755_b() {
+    public BiMap<Integer, Class> func_150755_b() {
         return this.field_150770_i;
     }
 
-    public BiMap func_150757_a(boolean p_150757_1_) {
+    public BiMap<Integer, Class> func_150757_a(boolean p_150757_1_) {
         return p_150757_1_ ? this.func_150755_b() : this.func_150753_a();
     }
 
-    public BiMap func_150754_b(boolean p_150754_1_) {
+    public BiMap<Integer, Class> func_150754_b(boolean p_150754_1_) {
         return p_150754_1_ ? this.func_150753_a() : this.func_150755_b();
     }
 
