--- a/net/minecraft/nbt/NBTTagCompound.java
+++ b/net/minecraft/nbt/NBTTagCompound.java
@@ -15,7 +15,7 @@
 
 public class NBTTagCompound extends NBTBase {
     private static final Logger field_150301_b = LogManager.getLogger();
-    private Map field_74784_a = new HashMap();
+    private Map<String, NBTBase> field_74784_a = new HashMap();
 
     @Override
     void func_74734_a(DataOutput p_74734_1_) throws IOException {
@@ -44,7 +44,7 @@
         }
     }
 
-    public Set func_150296_c() {
+    public Set<String> func_150296_c() {
         return this.field_74784_a.keySet();
     }
 
