--- a/net/minecraft/item/ItemStack.java
+++ b/net/minecraft/item/ItemStack.java
@@ -422,7 +422,7 @@
     @OnlyIn(Dist.CLIENT)
     public List<String> func_82840_a(EntityPlayer p_82840_1_, boolean p_82840_2_) {
         ArrayList arraylist = new ArrayList();
-        Object object = this.func_82833_r();
+        String object = this.func_82833_r();
         if (this.func_82837_s()) {
             object = EnumChatFormatting.ITALIC + object + EnumChatFormatting.RESET;
         }
@@ -479,7 +479,7 @@
             }
         }
 
-        Multimap multimap = this.func_111283_C();
+        Multimap<String, AttributeModifier> multimap = this.func_111283_C();
         if (!multimap.isEmpty()) {
             arraylist.add("");
 
@@ -601,7 +601,7 @@
     }
 
     public Multimap<String, AttributeModifier> func_111283_C() {
-        Object object;
+        Multimap<String, AttributeModifier> object;
         if (this.func_77942_o() && this.field_77990_d.func_150297_b("AttributeModifiers", 9)) {
             object = HashMultimap.create();
             NBTTagList nbttaglist = this.field_77990_d.func_150295_c("AttributeModifiers", 10);
