--- a/net/minecraft/world/chunk/storage/RegionFileCache.java
+++ b/net/minecraft/world/chunk/storage/RegionFileCache.java
@@ -8,7 +8,7 @@
 import java.util.Map;
 
 public class RegionFileCache {
-    private static final Map field_76553_a = new HashMap();
+    private static final Map<File, RegionFile> field_76553_a = new HashMap();
 
     public static synchronized RegionFile func_76550_a(File p_76550_0_, int p_76550_1_, int p_76550_2_) {
         File file1 = new File(p_76550_0_, "region");
