--- a/net/minecraft/world/chunk/storage/AnvilSaveConverter.java
+++ b/net/minecraft/world/chunk/storage/AnvilSaveConverter.java
@@ -115,7 +115,7 @@
         }
     }
 
-    private void func_75813_a(File p_75813_1_, Iterable p_75813_2_, WorldChunkManager p_75813_3_, int p_75813_4_, int p_75813_5_, IProgressUpdate p_75813_6_) {
+    private void func_75813_a(File p_75813_1_, Iterable<File> p_75813_2_, WorldChunkManager p_75813_3_, int p_75813_4_, int p_75813_5_, IProgressUpdate p_75813_6_) {
         for (File file1 : p_75813_2_) {
             this.func_75811_a(p_75813_1_, file1, p_75813_3_, p_75813_4_, p_75813_5_, p_75813_6_);
             p_75813_4_++;
