--- a/net/minecraft/client/resources/Language.java
+++ b/net/minecraft/client/resources/Language.java
@@ -4,7 +4,7 @@
 import net.neoforged.api.distmarker.OnlyIn;
 
 @OnlyIn(Dist.CLIENT)
-public class Language implements Comparable {
+public class Language implements Comparable<Language> {
     private final String field_135039_a;
     private final String field_135037_b;
     private final String field_135038_c;
