--- a/net/minecraft/client/resources/LanguageManager.java
+++ b/net/minecraft/client/resources/LanguageManager.java
@@ -22,7 +22,7 @@
     private final IMetadataSerializer field_135047_b;
     private String field_135048_c;
     protected static final Locale field_135049_a = new Locale();
-    private Map field_135046_d = Maps.newHashMap();
+    private Map<String, Language> field_135046_d = Maps.newHashMap();
 
     public LanguageManager(IMetadataSerializer p_i1304_1_, String p_i1304_2_) {
         this.field_135047_b = p_i1304_1_;
@@ -30,7 +30,7 @@
         I18n.func_135051_a(field_135049_a);
     }
 
-    public void func_135043_a(List p_135043_1_) {
+    public void func_135043_a(List<IResourcePack> p_135043_1_) {
         this.field_135046_d.clear();
 
         for (IResourcePack iresourcepack : p_135043_1_) {
@@ -80,7 +80,7 @@
             : (Language)this.field_135046_d.get("en_US");
     }
 
-    public SortedSet func_135040_d() {
+    public SortedSet<Language> func_135040_d() {
         return Sets.newTreeSet(this.field_135046_d.values());
     }
 }
