--- a/net/minecraft/entity/DataWatcher.java
+++ b/net/minecraft/entity/DataWatcher.java
@@ -19,7 +19,7 @@
     private final Entity field_151511_a;
     private boolean field_92086_a = true;
     private static final HashMap field_75697_a = new HashMap();
-    private final Map field_75695_b = new HashMap();
+    private final Map<Integer, DataWatcher.WatchableObject> field_75695_b = new HashMap();
     private boolean field_75696_c;
     private ReadWriteLock field_75694_d = new ReentrantReadWriteLock();
 
@@ -112,7 +112,7 @@
         return this.field_75696_c;
     }
 
-    public static void func_151507_a(List p_151507_0_, PacketBuffer p_151507_1_) throws IOException {
+    public static void func_151507_a(List<DataWatcher.WatchableObject> p_151507_0_, PacketBuffer p_151507_1_) throws IOException {
         if (p_151507_0_ != null) {
             for (DataWatcher.WatchableObject datawatcher$watchableobject : p_151507_0_) {
                 func_151510_a(p_151507_1_, datawatcher$watchableobject);
