--- a/net/minecraft/entity/EntityLiving.java
+++ b/net/minecraft/entity/EntityLiving.java
@@ -309,7 +309,7 @@
         super.func_70636_d();
         this.field_70170_p.field_72984_F.func_76320_a("looting");
         if (!this.field_70170_p.field_72995_K && this.func_98052_bS() && !this.field_70729_aU && this.field_70170_p.func_82736_K().func_82766_b("mobGriefing")) {
-            for (EntityItem entityitem : this.field_70170_p.func_72872_a(EntityItem.class, this.field_70121_D.func_72314_b(1.0, 0.0, 1.0))) {
+            for (EntityItem entityitem : (Iterable<EntityItem>)this.field_70170_p.func_72872_a(EntityItem.class, this.field_70121_D.func_72314_b(1.0, 0.0, 1.0))) {
                 if (!entityitem.field_70128_L && entityitem.func_92059_d() != null) {
                     ItemStack itemstack = entityitem.func_92059_d();
                     int i = func_82159_b(itemstack);
@@ -840,7 +840,7 @@
             if (this.field_110170_bx.func_150297_b("UUIDMost", 4) && this.field_110170_bx.func_150297_b("UUIDLeast", 4)) {
                 UUID uuid = new UUID(this.field_110170_bx.func_74763_f("UUIDMost"), this.field_110170_bx.func_74763_f("UUIDLeast"));
 
-                for (EntityLivingBase entitylivingbase : this.field_70170_p
+                for (EntityLivingBase entitylivingbase : (Iterable<EntityLivingBase>)this.field_70170_p
                     .func_72872_a(EntityLivingBase.class, this.field_70121_D.func_72314_b(10.0, 10.0, 10.0))) {
                     if (entitylivingbase.func_110124_au().equals(uuid)) {
                         this.field_110168_bw = entitylivingbase;
