--- a/net/minecraft/client/renderer/entity/RenderArrow.java
+++ b/net/minecraft/client/renderer/entity/RenderArrow.java
@@ -9,7 +9,7 @@
 import org.lwjgl.opengl.GL11;
 
 @OnlyIn(Dist.CLIENT)
-public class RenderArrow extends Render {
+public class RenderArrow extends Render<EntityArrow> {
     private static final ResourceLocation field_110780_a = new ResourceLocation("textures/entity/arrow.png");
 
     public void func_76986_a(EntityArrow p_76986_1_, double p_76986_2_, double p_76986_4_, double p_76986_6_, float p_76986_8_, float p_76986_9_) {
