--- a/net/minecraft/client/renderer/entity/RenderBoat.java
+++ b/net/minecraft/client/renderer/entity/RenderBoat.java
@@ -10,7 +10,7 @@
 import org.lwjgl.opengl.GL11;
 
 @OnlyIn(Dist.CLIENT)
-public class RenderBoat extends Render {
+public class RenderBoat extends Render<EntityBoat> {
     private static final ResourceLocation field_110782_f = new ResourceLocation("textures/entity/boat.png");
     protected ModelBase field_76998_a;
 
