--- a/net/minecraft/client/renderer/entity/RenderCaveSpider.java
+++ b/net/minecraft/client/renderer/entity/RenderCaveSpider.java
@@ -7,7 +7,7 @@
 import org.lwjgl.opengl.GL11;
 
 @OnlyIn(Dist.CLIENT)
-public class RenderCaveSpider extends RenderSpider {
+public class RenderCaveSpider extends RenderSpider<EntityCaveSpider> {
     private static final ResourceLocation field_110893_a = new ResourceLocation("textures/entity/spider/cave_spider.png");
 
     public RenderCaveSpider() {
