--- a/net/minecraft/client/renderer/entity/RenderGhast.java
+++ b/net/minecraft/client/renderer/entity/RenderGhast.java
@@ -8,7 +8,7 @@
 import org.lwjgl.opengl.GL11;
 
 @OnlyIn(Dist.CLIENT)
-public class RenderGhast extends RenderLiving {
+public class RenderGhast extends RenderLiving<EntityGhast> {
     private static final ResourceLocation field_110869_a = new ResourceLocation("textures/entity/ghast/ghast.png");
     private static final ResourceLocation field_110868_f = new ResourceLocation("textures/entity/ghast/ghast_shooting.png");
 
