--- a/net/minecraft/client/renderer/entity/RenderVillager.java
+++ b/net/minecraft/client/renderer/entity/RenderVillager.java
@@ -9,7 +9,7 @@
 import org.lwjgl.opengl.GL11;
 
 @OnlyIn(Dist.CLIENT)
-public class RenderVillager extends RenderLiving {
+public class RenderVillager extends RenderLiving<EntityVillager> {
     private static final ResourceLocation field_110903_f = new ResourceLocation("textures/entity/villager/villager.png");
     private static final ResourceLocation field_110904_g = new ResourceLocation("textures/entity/villager/farmer.png");
     private static final ResourceLocation field_110908_h = new ResourceLocation("textures/entity/villager/librarian.png");
@@ -27,7 +27,7 @@
     }
 
     public void func_76986_a(EntityVillager p_76986_1_, double p_76986_2_, double p_76986_4_, double p_76986_6_, float p_76986_8_, float p_76986_9_) {
-        super.func_76986_a((EntityLiving)p_76986_1_, p_76986_2_, p_76986_4_, p_76986_6_, p_76986_8_, p_76986_9_);
+        super.func_76986_a(p_76986_1_, p_76986_2_, p_76986_4_, p_76986_6_, p_76986_8_, p_76986_9_);
     }
 
     protected ResourceLocation func_110775_a(EntityVillager p_110775_1_) {
