--- a/net/minecraft/client/renderer/entity/RendererLivingEntity.java
+++ b/net/minecraft/client/renderer/entity/RendererLivingEntity.java
@@ -22,7 +22,7 @@
 import org.lwjgl.opengl.GL11;
 
 @OnlyIn(Dist.CLIENT)
-public abstract class RendererLivingEntity extends Render {
+public abstract class RendererLivingEntity<T extends EntityLivingBase> extends Render<T> {
     private static final Logger field_147923_a = LogManager.getLogger();
     private static final ResourceLocation field_110814_a = new ResourceLocation("textures/misc/enchanted_item_glint.png");
     protected ModelBase field_77045_g;
@@ -51,7 +51,7 @@
         return p_77034_1_ + p_77034_3_ * f;
     }
 
-    public void func_76986_a(EntityLivingBase p_76986_1_, double p_76986_2_, double p_76986_4_, double p_76986_6_, float p_76986_8_, float p_76986_9_) {
+    public void func_76986_a(T p_76986_1_, double p_76986_2_, double p_76986_4_, double p_76986_6_, float p_76986_8_, float p_76986_9_) {
         GL11.glPushMatrix();
         GL11.glDisable(2884);
         this.field_77045_g.field_78095_p = this.func_77040_d(p_76986_1_, p_76986_9_);
@@ -224,7 +224,7 @@
     }
 
     protected void func_77036_a(
-        EntityLivingBase p_77036_1_, float p_77036_2_, float p_77036_3_, float p_77036_4_, float p_77036_5_, float p_77036_6_, float p_77036_7_
+        T p_77036_1_, float p_77036_2_, float p_77036_3_, float p_77036_4_, float p_77036_5_, float p_77036_6_, float p_77036_7_
     ) {
         this.func_110777_b(p_77036_1_);
         if (!p_77036_1_.func_82150_aj()) {
@@ -246,11 +246,11 @@
         }
     }
 
-    protected void func_77039_a(EntityLivingBase p_77039_1_, double p_77039_2_, double p_77039_4_, double p_77039_6_) {
+    protected void func_77039_a(T p_77039_1_, double p_77039_2_, double p_77039_4_, double p_77039_6_) {
         GL11.glTranslatef((float)p_77039_2_, (float)p_77039_4_, (float)p_77039_6_);
     }
 
-    protected void func_77043_a(EntityLivingBase p_77043_1_, float p_77043_2_, float p_77043_3_, float p_77043_4_) {
+    protected void func_77043_a(T p_77043_1_, float p_77043_2_, float p_77043_3_, float p_77043_4_) {
         GL11.glRotatef(180.0F - p_77043_3_, 0.0F, 1.0F, 0.0F);
         if (p_77043_1_.field_70725_aQ > 0) {
             float f = (p_77043_1_.field_70725_aQ + p_77043_4_ - 1.0F) / 20.0F * 1.6F;
@@ -269,18 +269,18 @@
         }
     }
 
-    protected float func_77040_d(EntityLivingBase p_77040_1_, float p_77040_2_) {
+    protected float func_77040_d(T p_77040_1_, float p_77040_2_) {
         return p_77040_1_.func_70678_g(p_77040_2_);
     }
 
-    protected float func_77044_a(EntityLivingBase p_77044_1_, float p_77044_2_) {
+    protected float func_77044_a(T p_77044_1_, float p_77044_2_) {
         return p_77044_1_.field_70173_aa + p_77044_2_;
     }
 
-    protected void func_77029_c(EntityLivingBase p_77029_1_, float p_77029_2_) {
+    protected void func_77029_c(T p_77029_1_, float p_77029_2_) {
     }
 
-    protected void func_85093_e(EntityLivingBase p_85093_1_, float p_85093_2_) {
+    protected void func_85093_e(T p_85093_1_, float p_85093_2_) {
         int i = p_85093_1_.func_85035_bI();
         if (i > 0) {
             EntityArrow entityarrow = new EntityArrow(p_85093_1_.field_70170_p, p_85093_1_.field_70165_t, p_85093_1_.field_70163_u, p_85093_1_.field_70161_v);
@@ -320,29 +320,29 @@
         }
     }
 
-    protected int func_77035_b(EntityLivingBase p_77035_1_, int p_77035_2_, float p_77035_3_) {
+    protected int func_77035_b(T p_77035_1_, int p_77035_2_, float p_77035_3_) {
         return this.func_77032_a(p_77035_1_, p_77035_2_, p_77035_3_);
     }
 
-    protected int func_77032_a(EntityLivingBase p_77032_1_, int p_77032_2_, float p_77032_3_) {
+    protected int func_77032_a(T p_77032_1_, int p_77032_2_, float p_77032_3_) {
         return -1;
     }
 
-    protected void func_82408_c(EntityLivingBase p_82408_1_, int p_82408_2_, float p_82408_3_) {
+    protected void func_82408_c(T p_82408_1_, int p_82408_2_, float p_82408_3_) {
     }
 
-    protected float func_77037_a(EntityLivingBase p_77037_1_) {
+    protected float func_77037_a(T p_77037_1_) {
         return 90.0F;
     }
 
-    protected int func_77030_a(EntityLivingBase p_77030_1_, float p_77030_2_, float p_77030_3_) {
+    protected int func_77030_a(T p_77030_1_, float p_77030_2_, float p_77030_3_) {
         return 0;
     }
 
-    protected void func_77041_b(EntityLivingBase p_77041_1_, float p_77041_2_) {
+    protected void func_77041_b(T p_77041_1_, float p_77041_2_) {
     }
 
-    protected void func_77033_b(EntityLivingBase p_77033_1_, double p_77033_2_, double p_77033_4_, double p_77033_6_) {
+    protected void func_77033_b(T p_77033_1_, double p_77033_2_, double p_77033_4_, double p_77033_6_) {
         GL11.glAlphaFunc(516, 0.1F);
         if (this.func_110813_b(p_77033_1_)) {
             float f = 1.6F;
@@ -388,7 +388,7 @@
         }
     }
 
-    protected boolean func_110813_b(EntityLivingBase p_110813_1_) {
+    protected boolean func_110813_b(T p_110813_1_) {
         return Minecraft.func_71382_s()
             && p_110813_1_ != this.field_76990_c.field_78734_h
             && !p_110813_1_.func_98034_c(Minecraft.func_71410_x().field_71439_g)
@@ -396,7 +396,7 @@
     }
 
     protected void func_96449_a(
-        EntityLivingBase p_96449_1_, double p_96449_2_, double p_96449_4_, double p_96449_6_, String p_96449_8_, float p_96449_9_, double p_96449_10_
+        T p_96449_1_, double p_96449_2_, double p_96449_4_, double p_96449_6_, String p_96449_8_, float p_96449_9_, double p_96449_10_
     ) {
         if (p_96449_1_.func_70608_bn()) {
             this.func_147906_a(p_96449_1_, p_96449_8_, p_96449_2_, p_96449_4_ - 1.5, p_96449_6_, 64);
