--- a/net/minecraft/client/renderer/entity/RenderSheep.java
+++ b/net/minecraft/client/renderer/entity/RenderSheep.java
@@ -8,7 +8,7 @@
 import org.lwjgl.opengl.GL11;
 
 @OnlyIn(Dist.CLIENT)
-public class RenderSheep extends RenderLiving {
+public class RenderSheep extends RenderLiving<EntitySheep> {
     private static final ResourceLocation field_110885_a = new ResourceLocation("textures/entity/sheep/sheep_fur.png");
     private static final ResourceLocation field_110884_f = new ResourceLocation("textures/entity/sheep/sheep.png");
 
