--- a/net/minecraft/client/renderer/entity/RenderItem.java
+++ b/net/minecraft/client/renderer/entity/RenderItem.java
@@ -27,7 +27,7 @@
 import org.lwjgl.opengl.GL11;
 
 @OnlyIn(Dist.CLIENT)
-public class RenderItem extends Render {
+public class RenderItem extends Render<EntityItem> {
     private static final ResourceLocation field_110798_h = new ResourceLocation("textures/misc/enchanted_item_glint.png");
     private RenderBlocks field_147913_i = new RenderBlocks();
     private Random field_77025_h = new Random();
