--- a/net/minecraft/client/renderer/entity/RenderSkeleton.java
+++ b/net/minecraft/client/renderer/entity/RenderSkeleton.java
@@ -8,7 +8,7 @@
 import org.lwjgl.opengl.GL11;
 
 @OnlyIn(Dist.CLIENT)
-public class RenderSkeleton extends RenderBiped {
+public class RenderSkeleton extends RenderBiped<EntitySkeleton> {
     private static final ResourceLocation field_110862_k = new ResourceLocation("textures/entity/skeleton/skeleton.png");
     private static final ResourceLocation field_110861_l = new ResourceLocation("textures/entity/skeleton/wither_skeleton.png");
 
