--- a/net/minecraft/client/renderer/entity/RenderMagmaCube.java
+++ b/net/minecraft/client/renderer/entity/RenderMagmaCube.java
@@ -8,7 +8,7 @@
 import org.lwjgl.opengl.GL11;
 
 @OnlyIn(Dist.CLIENT)
-public class RenderMagmaCube extends RenderLiving {
+public class RenderMagmaCube extends RenderLiving<EntityMagmaCube> {
     private static final ResourceLocation field_110873_a = new ResourceLocation("textures/entity/slime/magmacube.png");
 
     public RenderMagmaCube() {
