--- a/net/minecraft/client/renderer/entity/RenderSpider.java
+++ b/net/minecraft/client/renderer/entity/RenderSpider.java
@@ -9,7 +9,7 @@
 import org.lwjgl.opengl.GL11;
 
 @OnlyIn(Dist.CLIENT)
-public class RenderSpider extends RenderLiving {
+public class RenderSpider<T extends EntitySpider> extends RenderLiving<T> {
     private static final ResourceLocation field_110891_a = new ResourceLocation("textures/entity/spider_eyes.png");
     private static final ResourceLocation field_110890_f = new ResourceLocation("textures/entity/spider/spider.png");
 
@@ -22,7 +22,7 @@
         return 180.0F;
     }
 
-    protected int func_77032_a(EntitySpider p_77032_1_, int p_77032_2_, float p_77032_3_) {
+    protected int func_77032_a(T p_77032_1_, int p_77032_2_, float p_77032_3_) {
         if (p_77032_2_ != 0) {
             return -1;
         } else {
@@ -45,7 +45,7 @@
         }
     }
 
-    protected ResourceLocation func_110775_a(EntitySpider p_110775_1_) {
+    protected ResourceLocation func_110775_a(T p_110775_1_) {
         return field_110890_f;
     }
 }
