--- a/net/minecraft/client/renderer/entity/RenderWitch.java
+++ b/net/minecraft/client/renderer/entity/RenderWitch.java
@@ -14,7 +14,7 @@
 import org.lwjgl.opengl.GL11;
 
 @OnlyIn(Dist.CLIENT)
-public class RenderWitch extends RenderLiving {
+public class RenderWitch extends RenderLiving<EntityWitch> {
     private static final ResourceLocation field_110910_a = new ResourceLocation("textures/entity/witch.png");
     private final ModelWitch field_82414_a = (ModelWitch)this.field_77045_g;
 
@@ -25,7 +25,7 @@
     public void func_76986_a(EntityWitch p_76986_1_, double p_76986_2_, double p_76986_4_, double p_76986_6_, float p_76986_8_, float p_76986_9_) {
         ItemStack itemstack = p_76986_1_.func_70694_bm();
         this.field_82414_a.field_82900_g = itemstack != null;
-        super.func_76986_a((EntityLiving)p_76986_1_, p_76986_2_, p_76986_4_, p_76986_6_, p_76986_8_, p_76986_9_);
+        super.func_76986_a(p_76986_1_, p_76986_2_, p_76986_4_, p_76986_6_, p_76986_8_, p_76986_9_);
     }
 
     protected ResourceLocation func_110775_a(EntityWitch p_110775_1_) {
