--- a/net/minecraft/client/renderer/entity/RenderPainting.java
+++ b/net/minecraft/client/renderer/entity/RenderPainting.java
@@ -10,7 +10,7 @@
 import org.lwjgl.opengl.GL11;
 
 @OnlyIn(Dist.CLIENT)
-public class RenderPainting extends Render {
+public class RenderPainting extends Render<EntityPainting> {
     private static final ResourceLocation field_110807_a = new ResourceLocation("textures/painting/paintings_kristoffer_zetterstrand.png");
 
     public void func_76986_a(EntityPainting p_76986_1_, double p_76986_2_, double p_76986_4_, double p_76986_6_, float p_76986_8_, float p_76986_9_) {
