--- a/net/minecraft/client/renderer/entity/RenderSquid.java
+++ b/net/minecraft/client/renderer/entity/RenderSquid.java
@@ -9,7 +9,7 @@
 import org.lwjgl.opengl.GL11;
 
 @OnlyIn(Dist.CLIENT)
-public class RenderSquid extends RenderLiving {
+public class RenderSquid extends RenderLiving<EntitySquid> {
     private static final ResourceLocation field_110901_a = new ResourceLocation("textures/entity/squid.png");
 
     public RenderSquid(ModelBase p_i1268_1_, float p_i1268_2_) {
@@ -17,7 +17,7 @@
     }
 
     public void func_76986_a(EntitySquid p_76986_1_, double p_76986_2_, double p_76986_4_, double p_76986_6_, float p_76986_8_, float p_76986_9_) {
-        super.func_76986_a((EntityLiving)p_76986_1_, p_76986_2_, p_76986_4_, p_76986_6_, p_76986_8_, p_76986_9_);
+        super.func_76986_a(p_76986_1_, p_76986_2_, p_76986_4_, p_76986_6_, p_76986_8_, p_76986_9_);
     }
 
     protected ResourceLocation func_110775_a(EntitySquid p_110775_1_) {
