--- a/net/minecraft/client/renderer/entity/RenderManager.java
+++ b/net/minecraft/client/renderer/entity/RenderManager.java
@@ -94,7 +94,7 @@
 
 @OnlyIn(Dist.CLIENT)
 public class RenderManager {
-    private Map field_78729_o = new HashMap();
+    private Map<Class, Render> field_78729_o = new HashMap();
     public static RenderManager field_78727_a = new RenderManager();
     private FontRenderer field_78736_p;
     public static double field_78725_b;
