--- a/net/minecraft/network/play/server/S00PacketKeepAlive.java
+++ b/net/minecraft/network/play/server/S00PacketKeepAlive.java
@@ -5,7 +5,7 @@
 import net.minecraft.network.PacketBuffer;
 import net.minecraft.network.play.INetHandlerPlayClient;
 
-public class S00PacketKeepAlive extends Packet {
+public class S00PacketKeepAlive extends Packet<INetHandlerPlayClient> {
     private int field_149136_a;
 
     public S00PacketKeepAlive() {
