--- a/net/minecraft/network/play/server/S01PacketJoinGame.java
+++ b/net/minecraft/network/play/server/S01PacketJoinGame.java
@@ -8,7 +8,7 @@
 import net.minecraft.world.WorldSettings;
 import net.minecraft.world.WorldType;
 
-public class S01PacketJoinGame extends Packet {
+public class S01PacketJoinGame extends Packet<INetHandlerPlayClient> {
     private int field_149206_a;
     private boolean field_149204_b;
     private WorldSettings.GameType field_149205_c;
