--- a/net/minecraft/network/play/server/S04PacketEntityEquipment.java
+++ b/net/minecraft/network/play/server/S04PacketEntityEquipment.java
@@ -6,7 +6,7 @@
 import net.minecraft.network.PacketBuffer;
 import net.minecraft.network.play.INetHandlerPlayClient;
 
-public class S04PacketEntityEquipment extends Packet {
+public class S04PacketEntityEquipment extends Packet<INetHandlerPlayClient> {
     private int field_149394_a;
     private int field_149392_b;
     private ItemStack field_149393_c;
