--- a/net/minecraft/network/play/server/S12PacketEntityVelocity.java
+++ b/net/minecraft/network/play/server/S12PacketEntityVelocity.java
@@ -6,7 +6,7 @@
 import net.minecraft.network.PacketBuffer;
 import net.minecraft.network.play.INetHandlerPlayClient;
 
-public class S12PacketEntityVelocity extends Packet {
+public class S12PacketEntityVelocity extends Packet<INetHandlerPlayClient> {
     private int field_149417_a;
     private int field_149415_b;
     private int field_149416_c;
