--- a/net/minecraft/network/play/server/S19PacketEntityStatus.java
+++ b/net/minecraft/network/play/server/S19PacketEntityStatus.java
@@ -7,7 +7,7 @@
 import net.minecraft.network.play.INetHandlerPlayClient;
 import net.minecraft.world.World;
 
-public class S19PacketEntityStatus extends Packet {
+public class S19PacketEntityStatus extends Packet<INetHandlerPlayClient> {
     private int field_149164_a;
     private byte field_149163_b;
 
