--- a/net/minecraft/network/play/server/S1EPacketRemoveEntityEffect.java
+++ b/net/minecraft/network/play/server/S1EPacketRemoveEntityEffect.java
@@ -6,7 +6,7 @@
 import net.minecraft.network.play.INetHandlerPlayClient;
 import net.minecraft.potion.PotionEffect;
 
-public class S1EPacketRemoveEntityEffect extends Packet {
+public class S1EPacketRemoveEntityEffect extends Packet<INetHandlerPlayClient> {
     private int field_149079_a;
     private int field_149078_b;
 
