--- a/net/minecraft/network/play/server/S20PacketEntityProperties.java
+++ b/net/minecraft/network/play/server/S20PacketEntityProperties.java
@@ -11,14 +11,14 @@
 import net.minecraft.network.PacketBuffer;
 import net.minecraft.network.play.INetHandlerPlayClient;
 
-public class S20PacketEntityProperties extends Packet {
+public class S20PacketEntityProperties extends Packet<INetHandlerPlayClient> {
     private int field_149445_a;
-    private final List field_149444_b = new ArrayList();
+    private final List<S20PacketEntityProperties.Snapshot> field_149444_b = new ArrayList();
 
     public S20PacketEntityProperties() {
     }
 
-    public S20PacketEntityProperties(int p_i45236_1_, Collection p_i45236_2_) {
+    public S20PacketEntityProperties(int p_i45236_1_, Collection<IAttributeInstance> p_i45236_2_) {
         this.field_149445_a = p_i45236_1_;
 
         for (IAttributeInstance iattributeinstance : p_i45236_2_) {
@@ -78,7 +78,7 @@
         return this.field_149445_a;
     }
 
-    public List func_149441_d() {
+    public List<S20PacketEntityProperties.Snapshot> func_149441_d() {
         return this.field_149444_b;
     }
 
@@ -101,7 +101,7 @@
             return this.field_151413_c;
         }
 
-        public Collection func_151408_c() {
+        public Collection<AttributeModifier> func_151408_c() {
             return this.field_151411_d;
         }
     }
