--- a/net/minecraft/network/play/server/S21PacketChunkData.java
+++ b/net/minecraft/network/play/server/S21PacketChunkData.java
@@ -11,7 +11,7 @@
 import net.minecraft.world.chunk.NibbleArray;
 import net.minecraft.world.chunk.storage.ExtendedBlockStorage;
 
-public class S21PacketChunkData extends Packet {
+public class S21PacketChunkData extends Packet<INetHandlerPlayClient> {
     private int field_149284_a;
     private int field_149282_b;
     private int field_149283_c;
