--- a/net/minecraft/network/play/server/S23PacketBlockChange.java
+++ b/net/minecraft/network/play/server/S23PacketBlockChange.java
@@ -7,7 +7,7 @@
 import net.minecraft.network.play.INetHandlerPlayClient;
 import net.minecraft.world.World;
 
-public class S23PacketBlockChange extends Packet {
+public class S23PacketBlockChange extends Packet<INetHandlerPlayClient> {
     private int field_148887_a;
     private int field_148885_b;
     private int field_148886_c;
