--- a/net/minecraft/network/play/server/S29PacketSoundEffect.java
+++ b/net/minecraft/network/play/server/S29PacketSoundEffect.java
@@ -6,7 +6,7 @@
 import net.minecraft.network.play.INetHandlerPlayClient;
 import org.apache.commons.lang3.Validate;
 
-public class S29PacketSoundEffect extends Packet {
+public class S29PacketSoundEffect extends Packet<INetHandlerPlayClient> {
     private String field_149219_a;
     private int field_149217_b;
     private int field_149218_c = Integer.MAX_VALUE;
