--- a/net/minecraft/network/play/server/S2DPacketOpenWindow.java
+++ b/net/minecraft/network/play/server/S2DPacketOpenWindow.java
@@ -5,7 +5,7 @@
 import net.minecraft.network.PacketBuffer;
 import net.minecraft.network.play.INetHandlerPlayClient;
 
-public class S2DPacketOpenWindow extends Packet {
+public class S2DPacketOpenWindow extends Packet<INetHandlerPlayClient> {
     private int field_148909_a;
     private int field_148907_b;
     private String field_148908_c;
