--- a/net/minecraft/network/play/server/S2EPacketCloseWindow.java
+++ b/net/minecraft/network/play/server/S2EPacketCloseWindow.java
@@ -5,7 +5,7 @@
 import net.minecraft.network.PacketBuffer;
 import net.minecraft.network.play.INetHandlerPlayClient;
 
-public class S2EPacketCloseWindow extends Packet {
+public class S2EPacketCloseWindow extends Packet<INetHandlerPlayClient> {
     private int field_148896_a;
 
     public S2EPacketCloseWindow() {
