--- a/net/minecraft/network/play/server/S2FPacketSetSlot.java
+++ b/net/minecraft/network/play/server/S2FPacketSetSlot.java
@@ -6,7 +6,7 @@
 import net.minecraft.network.PacketBuffer;
 import net.minecraft.network.play.INetHandlerPlayClient;
 
-public class S2FPacketSetSlot extends Packet {
+public class S2FPacketSetSlot extends Packet<INetHandlerPlayClient> {
     private int field_149179_a;
     private int field_149177_b;
     private ItemStack field_149178_c;
