--- a/net/minecraft/network/play/server/S32PacketConfirmTransaction.java
+++ b/net/minecraft/network/play/server/S32PacketConfirmTransaction.java
@@ -5,7 +5,7 @@
 import net.minecraft.network.PacketBuffer;
 import net.minecraft.network.play.INetHandlerPlayClient;
 
-public class S32PacketConfirmTransaction extends Packet {
+public class S32PacketConfirmTransaction extends Packet<INetHandlerPlayClient> {
     private int field_148894_a;
     private short field_148892_b;
     private boolean field_148893_c;
