--- a/net/minecraft/network/play/server/S33PacketUpdateSign.java
+++ b/net/minecraft/network/play/server/S33PacketUpdateSign.java
@@ -5,7 +5,7 @@
 import net.minecraft.network.PacketBuffer;
 import net.minecraft.network.play.INetHandlerPlayClient;
 
-public class S33PacketUpdateSign extends Packet {
+public class S33PacketUpdateSign extends Packet<INetHandlerPlayClient> {
     private int field_149352_a;
     private int field_149350_b;
     private int field_149351_c;
