--- a/net/minecraft/network/play/server/S40PacketDisconnect.java
+++ b/net/minecraft/network/play/server/S40PacketDisconnect.java
@@ -6,7 +6,7 @@
 import net.minecraft.network.play.INetHandlerPlayClient;
 import net.minecraft.util.IChatComponent;
 
-public class S40PacketDisconnect extends Packet {
+public class S40PacketDisconnect extends Packet<INetHandlerPlayClient> {
     private IChatComponent field_149167_a;
 
     public S40PacketDisconnect() {
