--- a/net/minecraft/client/resources/data/BaseMetadataSectionSerializer.java
+++ b/net/minecraft/client/resources/data/BaseMetadataSectionSerializer.java
@@ -4,5 +4,5 @@
 import net.neoforged.api.distmarker.OnlyIn;
 
 @OnlyIn(Dist.CLIENT)
-public abstract class BaseMetadataSectionSerializer implements IMetadataSectionSerializer {
+public abstract class BaseMetadataSectionSerializer<T> implements IMetadataSectionSerializer<T> {
 }
