--- a/net/minecraft/client/resources/data/IMetadataSectionSerializer.java
+++ b/net/minecraft/client/resources/data/IMetadataSectionSerializer.java
@@ -5,6 +5,6 @@
 import net.neoforged.api.distmarker.OnlyIn;
 
 @OnlyIn(Dist.CLIENT)
-public interface IMetadataSectionSerializer extends JsonDeserializer {
+public interface IMetadataSectionSerializer<T> extends JsonDeserializer<T> {
     String func_110483_a();
 }
