--- a/net/minecraft/network/handshake/client/C00Handshake.java
+++ b/net/minecraft/network/handshake/client/C00Handshake.java
@@ -8,7 +8,7 @@
 import net.neoforged.api.distmarker.Dist;
 import net.neoforged.api.distmarker.OnlyIn;
 
-public class C00Handshake extends Packet {
+public class C00Handshake extends Packet<INetHandlerHandshakeServer> {
     private int field_149600_a;
     private String field_149598_b;
     private int field_149599_c;
