--- a/net/minecraft/client/network/LanServerDetector.java
+++ b/net/minecraft/client/network/LanServerDetector.java
@@ -43,7 +43,7 @@
     }
 
     public static class LanServerList {
-        private ArrayList field_77555_b = new ArrayList();
+        private ArrayList<LanServerDetector.LanServer> field_77555_b = new ArrayList();
         boolean field_77556_a;
 
         public synchronized boolean func_77553_a() {
@@ -54,7 +54,7 @@
             this.field_77556_a = false;
         }
 
-        public synchronized List func_77554_c() {
+        public synchronized List<LanServerDetector.LanServer> func_77554_c() {
             return Collections.unmodifiableList(this.field_77555_b);
         }
 
