--- a/net/minecraft/client/network/OldServerPinger.java
+++ b/net/minecraft/client/network/OldServerPinger.java
@@ -217,10 +217,10 @@
                                         }
                                     }
 
-                                    protected void channelRead0(ChannelHandlerContext p_channelRead0_1_, ByteBuf p_channelRead0_2_) {
-                                        short short1 = p_channelRead0_2_.readUnsignedByte();
+                                    protected void channelRead0(ChannelHandlerContext p_channelRead0_1_, Object p_channelRead0_2_) {
+                                        short short1 = ((ByteBuf)p_channelRead0_2_).readUnsignedByte();
                                         if (short1 == 255) {
-                                            String s = new String(p_channelRead0_2_.readBytes(p_channelRead0_2_.readShort() * 2).array(), Charsets.UTF_16BE);
+                                            String s = new String(((ByteBuf)p_channelRead0_2_).readBytes(((ByteBuf)p_channelRead0_2_).readShort() * 2).array(), Charsets.UTF_16BE);
                                             String[] astring = Iterables.toArray(OldServerPinger.field_147230_a.split(s), String.class);
                                             if ("\u00a71".equals(astring[0])) {
                                                 int i = MathHelper.func_82715_a(astring[1], 0);
