--- a/net/minecraft/network/login/server/S00PacketDisconnect.java
+++ b/net/minecraft/network/login/server/S00PacketDisconnect.java
@@ -8,7 +8,7 @@
 import net.neoforged.api.distmarker.Dist;
 import net.neoforged.api.distmarker.OnlyIn;
 
-public class S00PacketDisconnect extends Packet {
+public class S00PacketDisconnect extends Packet<INetHandlerLoginClient> {
     private IChatComponent field_149605_a;
 
     public S00PacketDisconnect() {
