--- a/net/minecraft/entity/EntityList.java
+++ b/net/minecraft/entity/EntityList.java
@@ -68,12 +68,12 @@
 
 public class EntityList {
     private static final Logger field_151516_b = LogManager.getLogger();
-    private static Map field_75625_b = new HashMap();
-    private static Map field_75626_c = new HashMap();
-    private static Map field_75623_d = new HashMap();
-    private static Map field_75624_e = new HashMap();
-    private static Map field_75622_f = new HashMap();
-    public static HashMap field_75627_a = new LinkedHashMap();
+    private static Map<String, Class> field_75625_b = new HashMap();
+    private static Map<Class, String> field_75626_c = new HashMap();
+    private static Map<Integer, Class> field_75623_d = new HashMap();
+    private static Map<Class, Integer> field_75624_e = new HashMap();
+    private static Map<String, Integer> field_75622_f = new HashMap();
+    public static HashMap<Integer, EntityList.EntityEggInfo> field_75627_a = new LinkedHashMap();
 
     private static void func_75618_a(Class p_75618_0_, String p_75618_1_, int p_75618_2_) {
         if (field_75625_b.containsKey(p_75618_1_)) {
@@ -184,7 +184,7 @@
     public static void func_151514_a() {
     }
 
-    public static Set func_151515_b() {
+    public static Set<String> func_151515_b() {
         return Collections.unmodifiableSet(field_75622_f.keySet());
     }
 
