--- a/net/minecraft/entity/ai/EntityAITasks.java
+++ b/net/minecraft/entity/ai/EntityAITasks.java
@@ -41,7 +41,7 @@
     }
 
     public void func_75774_a() {
-        ArrayList arraylist = new ArrayList();
+        ArrayList<EntityAITasks.EntityAITaskEntry> arraylist = new ArrayList();
         if (this.field_75778_d++ % this.field_75779_e == 0) {
             for (EntityAITasks.EntityAITaskEntry entityaitasks$entityaitaskentry : this.field_75782_a) {
                 boolean flag = this.field_75780_b.contains(entityaitasks$entityaitaskentry);
