--- a/net/minecraft/entity/ai/EntityAINearestAttackableTarget.java
+++ b/net/minecraft/entity/ai/EntityAINearestAttackableTarget.java
@@ -70,7 +70,7 @@
         super.func_75249_e();
     }
 
-    public static class Sorter implements Comparator {
+    public static class Sorter implements Comparator<Entity> {
         private final Entity field_75459_b;
 
         public Sorter(Entity p_i1662_1_) {
