--- a/net/minecraft/world/Explosion.java
+++ b/net/minecraft/world/Explosion.java
@@ -30,7 +30,7 @@
     public double field_77282_d;
     public Entity field_77283_e;
     public float field_77280_f;
-    public List field_77281_g = new ArrayList();
+    public List<ChunkPosition> field_77281_g = new ArrayList();
     private Map field_77288_k = new HashMap();
 
     public Explosion(World p_i1948_1_, Entity p_i1948_2_, double p_i1948_3_, double p_i1948_5_, double p_i1948_7_, float p_i1948_9_) {
