--- a/net/minecraft/world/World.java
+++ b/net/minecraft/world/World.java
@@ -2274,12 +2274,12 @@
         return null;
     }
 
-    public List func_72839_b(Entity p_72839_1_, AxisAlignedBB p_72839_2_) {
+    public List<Entity> func_72839_b(Entity p_72839_1_, AxisAlignedBB p_72839_2_) {
         return this.func_94576_a(p_72839_1_, p_72839_2_, null);
     }
 
-    public List func_94576_a(Entity p_94576_1_, AxisAlignedBB p_94576_2_, IEntitySelector p_94576_3_) {
-        ArrayList arraylist = new ArrayList();
+    public List<Entity> func_94576_a(Entity p_94576_1_, AxisAlignedBB p_94576_2_, IEntitySelector p_94576_3_) {
+        ArrayList<Entity> arraylist = new ArrayList();
         int i = MathHelper.func_76128_c((p_94576_2_.field_72340_a - 2.0) / 16.0);
         int j = MathHelper.func_76128_c((p_94576_2_.field_72336_d + 2.0) / 16.0);
         int k = MathHelper.func_76128_c((p_94576_2_.field_72339_c - 2.0) / 16.0);
