--- a/net/minecraft/server/management/PreYggdrasilConverter.java
+++ b/net/minecraft/server/management/PreYggdrasilConverter.java
@@ -24,8 +24,8 @@
     public static final File field_152731_d = new File("white-list.txt");
 
     private static void func_152717_a(MinecraftServer p_152717_0_, Collection p_152717_1_, ProfileLookupCallback p_152717_2_) {
-        String[] astring = Iterators.toArray(Iterators.filter(p_152717_1_.iterator(), new Predicate() {
-            public boolean func_152733_a(String p_152733_1_) {
+        String[] astring = Iterators.toArray(Iterators.filter(p_152717_1_.iterator(), new Predicate<String>() {
+            public boolean apply(String p_152733_1_) {
                 return !StringUtils.func_151246_b(p_152733_1_);
             }
         }), String.class);
