--- a/net/minecraft/server/management/ServerConfigurationManager.java
+++ b/net/minecraft/server/management/ServerConfigurationManager.java
@@ -292,7 +292,7 @@
 
     public EntityPlayerMP func_148545_a(GameProfile p_148545_1_) {
         UUID uuid = EntityPlayer.func_146094_a(p_148545_1_);
-        ArrayList arraylist = Lists.newArrayList();
+        ArrayList<EntityPlayerMP> arraylist = Lists.newArrayList();
 
         for (int i = 0; i < this.field_72404_b.size(); i++) {
             EntityPlayerMP entityplayermp = this.field_72404_b.get(i);
@@ -593,7 +593,7 @@
         if (this.field_72404_b.isEmpty()) {
             return Collections.emptyList();
         } else {
-            Object object = new ArrayList();
+            List object = new ArrayList();
             boolean flag = p_82449_4_ < 0;
             boolean flag1 = p_82449_9_ != null && p_82449_9_.startsWith("!");
             boolean flag2 = p_82449_10_ != null && p_82449_10_.startsWith("!");
