--- a/net/minecraft/server/management/UserListWhitelist.java
+++ b/net/minecraft/server/management/UserListWhitelist.java
@@ -19,7 +19,7 @@
         String[] astring = new String[this.func_152688_e().size()];
         int i = 0;
 
-        for (UserListWhitelistEntry userlistwhitelistentry : this.func_152688_e().values()) {
+        for (UserListWhitelistEntry userlistwhitelistentry : (Iterable<UserListWhitelistEntry>) (Iterable) this.func_152688_e().values()) {
             astring[i++] = ((GameProfile)userlistwhitelistentry.func_152640_f()).getName();
         }
 
@@ -31,7 +31,7 @@
     }
 
     public GameProfile func_152706_a(String p_152706_1_) {
-        for (UserListWhitelistEntry userlistwhitelistentry : this.func_152688_e().values()) {
+        for (UserListWhitelistEntry userlistwhitelistentry : (Iterable<UserListWhitelistEntry>) (Iterable) this.func_152688_e().values()) {
             if (p_152706_1_.equalsIgnoreCase(((GameProfile)userlistwhitelistentry.func_152640_f()).getName())) {
                 return (GameProfile)userlistwhitelistentry.func_152640_f();
             }
