--- a/net/minecraft/network/login/server/S01PacketEncryptionRequest.java
+++ b/net/minecraft/network/login/server/S01PacketEncryptionRequest.java
@@ -7,7 +7,7 @@
 import net.minecraft.network.login.INetHandlerLoginClient;
 import net.minecraft.util.CryptManager;
 
-public class S01PacketEncryptionRequest extends Packet {
+public class S01PacketEncryptionRequest extends Packet<INetHandlerLoginClient> {
     private String field_149612_a;
     private PublicKey field_149610_b;
     private byte[] field_149611_c;
