--- a/net/minecraft/item/crafting/CraftingManager.java
+++ b/net/minecraft/item/crafting/CraftingManager.java
@@ -15,7 +15,7 @@
 
 public class CraftingManager {
     private static final CraftingManager field_77598_a = new CraftingManager();
-    private List field_77597_b = new ArrayList();
+    private List<IRecipe> field_77597_b = new ArrayList();
 
     public static final CraftingManager func_77594_a() {
         return field_77598_a;
@@ -192,7 +192,7 @@
             new ItemStack(Blocks.field_150453_bW), "GGG", "QQQ", "WWW", 'G', Blocks.field_150359_w, 'Q', Items.field_151128_bU, 'W', Blocks.field_150376_bx
         );
         this.func_92103_a(new ItemStack(Blocks.field_150438_bZ), "I I", "ICI", " I ", 'I', Items.field_151042_j, 'C', Blocks.field_150486_ae);
-        Collections.sort(this.field_77597_b, new Comparator() {
+        Collections.sort(this.field_77597_b, new Comparator<IRecipe>() {
             public int compare(IRecipe p_compare_1_, IRecipe p_compare_2_) {
                 if (p_compare_1_ instanceof ShapelessRecipes && p_compare_2_ instanceof ShapedRecipes) {
                     return 1;
@@ -328,7 +328,7 @@
         }
     }
 
-    public List func_77592_b() {
+    public List<IRecipe> func_77592_b() {
         return this.field_77597_b;
     }
 }
