--- a/net/minecraft/client/gui/stream/GuiStreamUnavailable.java
+++ b/net/minecraft/client/gui/stream/GuiStreamUnavailable.java
@@ -30,8 +30,8 @@
     private final IChatComponent field_152324_f = new ChatComponentTranslation("stream.unavailable.title");
     private final GuiScreen field_152325_g;
     private final GuiStreamUnavailable.Reason field_152326_h;
-    private final List field_152327_i;
-    private final List field_152323_r = Lists.newArrayList();
+    private final List<ChatComponentTranslation> field_152327_i;
+    private final List<String> field_152323_r = Lists.newArrayList();
 
     public GuiStreamUnavailable(GuiScreen p_i1070_1_, GuiStreamUnavailable.Reason p_i1070_2_) {
         this(p_i1070_1_, p_i1070_2_, null);
