--- a/net/minecraft/network/play/server/S07PacketRespawn.java
+++ b/net/minecraft/network/play/server/S07PacketRespawn.java
@@ -8,7 +8,7 @@
 import net.minecraft.world.WorldSettings;
 import net.minecraft.world.WorldType;
 
-public class S07PacketRespawn extends Packet {
+public class S07PacketRespawn extends Packet<INetHandlerPlayClient> {
     private int field_149088_a;
     private EnumDifficulty field_149086_b;
     private WorldSettings.GameType field_149087_c;
