--- a/net/minecraft/network/play/server/S0DPacketCollectItem.java
+++ b/net/minecraft/network/play/server/S0DPacketCollectItem.java
@@ -5,7 +5,7 @@
 import net.minecraft.network.PacketBuffer;
 import net.minecraft.network.play.INetHandlerPlayClient;
 
-public class S0DPacketCollectItem extends Packet {
+public class S0DPacketCollectItem extends Packet<INetHandlerPlayClient> {
     private int field_149357_a;
     private int field_149356_b;
 
