--- a/net/minecraft/network/play/server/S13PacketDestroyEntities.java
+++ b/net/minecraft/network/play/server/S13PacketDestroyEntities.java
@@ -5,7 +5,7 @@
 import net.minecraft.network.PacketBuffer;
 import net.minecraft.network.play.INetHandlerPlayClient;
 
-public class S13PacketDestroyEntities extends Packet {
+public class S13PacketDestroyEntities extends Packet<INetHandlerPlayClient> {
     private int[] field_149100_a;
 
     public S13PacketDestroyEntities() {
