--- a/net/minecraft/network/play/server/S14PacketEntity.java
+++ b/net/minecraft/network/play/server/S14PacketEntity.java
@@ -5,7 +5,7 @@
 import net.minecraft.network.PacketBuffer;
 import net.minecraft.network.play.INetHandlerPlayClient;
 
-public class S14PacketEntity extends Packet {
+public class S14PacketEntity extends Packet<INetHandlerPlayClient> {
     protected int field_149074_a;
     protected byte field_149072_b;
     protected byte field_149073_c;
