--- a/net/minecraft/network/play/server/S18PacketEntityTeleport.java
+++ b/net/minecraft/network/play/server/S18PacketEntityTeleport.java
@@ -7,7 +7,7 @@
 import net.minecraft.network.play.INetHandlerPlayClient;
 import net.minecraft.util.MathHelper;
 
-public class S18PacketEntityTeleport extends Packet {
+public class S18PacketEntityTeleport extends Packet<INetHandlerPlayClient> {
     private int field_149458_a;
     private int field_149456_b;
     private int field_149457_c;
