--- a/net/minecraft/network/play/server/S22PacketMultiBlockChange.java
+++ b/net/minecraft/network/play/server/S22PacketMultiBlockChange.java
@@ -12,7 +12,7 @@
 import org.apache.logging.log4j.LogManager;
 import org.apache.logging.log4j.Logger;
 
-public class S22PacketMultiBlockChange extends Packet {
+public class S22PacketMultiBlockChange extends Packet<INetHandlerPlayClient> {
     private static final Logger field_148927_a = LogManager.getLogger();
     private ChunkCoordIntPair field_148925_b;
     private byte[] field_148926_c;
