--- a/net/minecraft/network/play/server/S2APacketParticles.java
+++ b/net/minecraft/network/play/server/S2APacketParticles.java
@@ -5,7 +5,7 @@
 import net.minecraft.network.PacketBuffer;
 import net.minecraft.network.play.INetHandlerPlayClient;
 
-public class S2APacketParticles extends Packet {
+public class S2APacketParticles extends Packet<INetHandlerPlayClient> {
     private String field_149236_a;
     private float field_149234_b;
     private float field_149235_c;
