--- a/net/minecraft/network/play/server/S3APacketTabComplete.java
+++ b/net/minecraft/network/play/server/S3APacketTabComplete.java
@@ -6,7 +6,7 @@
 import net.minecraft.network.play.INetHandlerPlayClient;
 import org.apache.commons.lang3.ArrayUtils;
 
-public class S3APacketTabComplete extends Packet {
+public class S3APacketTabComplete extends Packet<INetHandlerPlayClient> {
     private String[] field_149632_a;
 
     public S3APacketTabComplete() {
