--- a/net/minecraft/network/play/server/S3FPacketCustomPayload.java
+++ b/net/minecraft/network/play/server/S3FPacketCustomPayload.java
@@ -6,7 +6,7 @@
 import net.minecraft.network.PacketBuffer;
 import net.minecraft.network.play.INetHandlerPlayClient;
 
-public class S3FPacketCustomPayload extends Packet {
+public class S3FPacketCustomPayload extends Packet<INetHandlerPlayClient> {
     private String field_149172_a;
     private byte[] field_149171_b;
 
