--- a/net/minecraft/server/management/PlayerPositionComparator.java
+++ b/net/minecraft/server/management/PlayerPositionComparator.java
@@ -4,7 +4,7 @@
 import net.minecraft.entity.player.EntityPlayerMP;
 import net.minecraft.util.ChunkCoordinates;
 
-public class PlayerPositionComparator implements Comparator {
+public class PlayerPositionComparator implements Comparator<EntityPlayerMP> {
     private final ChunkCoordinates field_82548_a;
 
     public PlayerPositionComparator(ChunkCoordinates p_i1499_1_) {
