--- a/net/minecraft/client/renderer/EntitySorter.java
+++ b/net/minecraft/client/renderer/EntitySorter.java
@@ -3,7 +3,7 @@
 import java.util.Comparator;
 import net.minecraft.entity.Entity;
 
-public class EntitySorter implements Comparator {
+public class EntitySorter implements Comparator<WorldRenderer> {
     private double field_78949_a;
     private double field_78947_b;
     private double field_78948_c;
