--- a/net/minecraft/client/renderer/RenderSorter.java
+++ b/net/minecraft/client/renderer/RenderSorter.java
@@ -3,7 +3,7 @@
 import java.util.Comparator;
 import net.minecraft.entity.EntityLivingBase;
 
-public class RenderSorter implements Comparator {
+public class RenderSorter implements Comparator<WorldRenderer> {
     private EntityLivingBase field_78945_a;
 
     public RenderSorter(EntityLivingBase p_i1241_1_) {
