--- a/net/minecraft/tileentity/MobSpawnerBaseLogic.java
+++ b/net/minecraft/tileentity/MobSpawnerBaseLogic.java
@@ -16,7 +16,7 @@
 public abstract class MobSpawnerBaseLogic {
     public int field_98286_b = 20;
     private String field_98288_a = "Pig";
-    private List field_98285_e;
+    private List<MobSpawnerBaseLogic.WeightedRandomMinecart> field_98285_e;
     private MobSpawnerBaseLogic.WeightedRandomMinecart field_98282_f;
     public double field_98287_c;
     public double field_98284_d;
