--- a/net/minecraft/tileentity/TileEntityChest.java
+++ b/net/minecraft/tileentity/TileEntityChest.java
@@ -237,7 +237,7 @@
             this.field_145987_o = 0;
             float f = 5.0F;
 
-            for (EntityPlayer entityplayer : this.field_145850_b
+            for (EntityPlayer entityplayer : (Iterable<EntityPlayer>)this.field_145850_b
                 .func_72872_a(
                     EntityPlayer.class,
                     AxisAlignedBB.func_72330_a(
