--- a/net/minecraft/stats/StatisticsFile.java
+++ b/net/minecraft/stats/StatisticsFile.java
@@ -74,7 +74,7 @@
         }
     }
 
-    public Set func_150878_c() {
+    public Set<StatBase> func_150878_c() {
         HashSet hashset = Sets.newHashSet(this.field_150888_e);
         this.field_150888_e.clear();
         this.field_150886_g = false;
@@ -123,7 +123,7 @@
         }
     }
 
-    public static String func_150880_a(Map p_150880_0_) {
+    public static String func_150880_a(Map<StatBase, TupleIntJsonSerializable> p_150880_0_) {
         JsonObject jsonobject = new JsonObject();
 
         for (Entry entry : p_150880_0_.entrySet()) {
