--- a/net/minecraft/realms/RealmsLevelSummary.java
+++ b/net/minecraft/realms/RealmsLevelSummary.java
@@ -2,7 +2,7 @@
 
 import net.minecraft.world.storage.SaveFormatComparator;
 
-public class RealmsLevelSummary implements Comparable {
+public class RealmsLevelSummary implements Comparable<RealmsLevelSummary> {
     private SaveFormatComparator levelSummary;
 
     public RealmsLevelSummary(SaveFormatComparator p_i1109_1_) {
