--- a/net/minecraft/client/audio/SoundList.java
+++ b/net/minecraft/client/audio/SoundList.java
@@ -7,11 +7,11 @@
 
 @OnlyIn(Dist.CLIENT)
 public class SoundList {
-    private final List field_148577_a = Lists.newArrayList();
+    private final List<SoundList.SoundEntry> field_148577_a = Lists.newArrayList();
     private boolean field_148575_b;
     private SoundCategory field_148576_c;
 
-    public List func_148570_a() {
+    public List<SoundList.SoundEntry> func_148570_a() {
         return this.field_148577_a;
     }
 
