--- a/net/minecraft/client/renderer/tileentity/RenderEnderCrystal.java
+++ b/net/minecraft/client/renderer/tileentity/RenderEnderCrystal.java
@@ -11,7 +11,7 @@
 import org.lwjgl.opengl.GL11;
 
 @OnlyIn(Dist.CLIENT)
-public class RenderEnderCrystal extends Render {
+public class RenderEnderCrystal extends Render<EntityEnderCrystal> {
     private static final ResourceLocation field_110787_a = new ResourceLocation("textures/entity/endercrystal/endercrystal.png");
     private ModelBase field_76995_b;
 
