--- a/net/minecraft/client/renderer/tileentity/RenderWitherSkull.java
+++ b/net/minecraft/client/renderer/tileentity/RenderWitherSkull.java
@@ -9,7 +9,7 @@
 import org.lwjgl.opengl.GL11;
 
 @OnlyIn(Dist.CLIENT)
-public class RenderWitherSkull extends Render {
+public class RenderWitherSkull extends Render<EntityWitherSkull> {
     private static final ResourceLocation field_110811_a = new ResourceLocation("textures/entity/wither/wither_invulnerable.png");
     private static final ResourceLocation field_110810_f = new ResourceLocation("textures/entity/wither/wither.png");
     private final ModelSkeletonHead field_82401_a = new ModelSkeletonHead();
