--- a/net/minecraft/client/renderer/tileentity/TileEntitySpecialRenderer.java
+++ b/net/minecraft/client/renderer/tileentity/TileEntitySpecialRenderer.java
@@ -9,10 +9,10 @@
 import net.neoforged.api.distmarker.OnlyIn;
 
 @OnlyIn(Dist.CLIENT)
-public abstract class TileEntitySpecialRenderer {
+public abstract class TileEntitySpecialRenderer<T extends TileEntity> {
     protected TileEntityRendererDispatcher field_147501_a;
 
-    public abstract void func_147500_a(TileEntity var1, double var2, double var4, double var6, float var8);
+    public abstract void func_147500_a(T var1, double var2, double var4, double var6, float var8);
 
     protected void func_147499_a(ResourceLocation p_147499_1_) {
         TextureManager texturemanager = this.field_147501_a.field_147553_e;
