--- a/net/minecraft/client/renderer/tileentity/TileEntityChestRenderer.java
+++ b/net/minecraft/client/renderer/tileentity/TileEntityChestRenderer.java
@@ -12,7 +12,7 @@
 import org.lwjgl.opengl.GL11;
 
 @OnlyIn(Dist.CLIENT)
-public class TileEntityChestRenderer extends TileEntitySpecialRenderer {
+public class TileEntityChestRenderer extends TileEntitySpecialRenderer<TileEntityChest> {
     private static final ResourceLocation field_147507_b = new ResourceLocation("textures/entity/chest/trapped_double.png");
     private static final ResourceLocation field_147508_c = new ResourceLocation("textures/entity/chest/christmas_double.png");
     private static final ResourceLocation field_147505_d = new ResourceLocation("textures/entity/chest/normal_double.png");
