--- a/net/minecraft/client/multiplayer/ServerList.java
+++ b/net/minecraft/client/multiplayer/ServerList.java
@@ -16,7 +16,7 @@
 public class ServerList {
     private static final Logger field_147415_a = LogManager.getLogger();
     private final Minecraft field_78859_a;
-    private final List field_78858_b = new ArrayList();
+    private final List<ServerData> field_78858_b = new ArrayList();
 
     public ServerList(Minecraft p_i1194_1_) {
         this.field_78859_a = p_i1194_1_;
