--- a/net/minecraft/item/ItemPotion.java
+++ b/net/minecraft/item/ItemPotion.java
@@ -69,7 +69,7 @@
         }
 
         if (!p_77654_2_.field_72995_K) {
-            List list = this.func_77832_l(p_77654_1_);
+            List<PotionEffect> list = this.func_77832_l(p_77654_1_);
             if (list != null) {
                 for (PotionEffect potioneffect : list) {
                     p_77654_3_.func_70690_d(new PotionEffect(potioneffect));
