--- a/net/minecraft/item/ItemStack.java
+++ b/net/minecraft/item/ItemStack.java
@@ -464,7 +464,7 @@
     }
 
     public Multimap<String, AttributeModifier> func_111283_C() {
-        Object object;
+        Multimap<String, AttributeModifier> object;
         if (this.func_77942_o() && this.field_77990_d.func_150297_b("AttributeModifiers", 9)) {
             object = HashMultimap.create();
             NBTTagList nbttaglist = this.field_77990_d.func_150295_c("AttributeModifiers", 10);
