--- a/net/minecraft/item/ItemPotion.java
+++ b/net/minecraft/item/ItemPotion.java
@@ -80,7 +80,7 @@
         }
 
         if (!p_77654_2_.field_72995_K) {
-            List list = this.func_77832_l(p_77654_1_);
+            List<PotionEffect> list = this.func_77832_l(p_77654_1_);
             if (list != null) {
                 for (PotionEffect potioneffect : list) {
                     p_77654_3_.func_70690_d(new PotionEffect(potioneffect));
@@ -173,7 +173,7 @@
     }
 
     public boolean func_77833_h(int p_77833_1_) {
-        List list = this.func_77834_f(p_77833_1_);
+        List<PotionEffect> list = this.func_77834_f(p_77833_1_);
         if (list != null && !list.isEmpty()) {
             for (PotionEffect potioneffect : list) {
                 if (Potion.field_76425_a[potioneffect.func_76456_a()].func_76403_b()) {
@@ -212,13 +212,13 @@
     @Override
     public void func_77624_a(ItemStack p_77624_1_, EntityPlayer p_77624_2_, List p_77624_3_, boolean p_77624_4_) {
         if (p_77624_1_.func_77960_j() != 0) {
-            List list = Items.field_151068_bn.func_77832_l(p_77624_1_);
-            HashMultimap hashmultimap = HashMultimap.create();
+            List<PotionEffect> list = Items.field_151068_bn.func_77832_l(p_77624_1_);
+            HashMultimap<String, AttributeModifier> hashmultimap = HashMultimap.create();
             if (list != null && !list.isEmpty()) {
                 for (PotionEffect potioneffect : list) {
-                    Object object1 = StatCollector.func_74838_a(potioneffect.func_76453_d()).trim();
+                    String object1 = StatCollector.func_74838_a(potioneffect.func_76453_d()).trim();
                     Potion potion = Potion.field_76425_a[potioneffect.func_76456_a()];
-                    Map map = potion.func_111186_k();
+                    Map<IAttribute, AttributeModifier> map = potion.func_111186_k();
                     if (map != null && map.size() > 0) {
                         for (Entry entry : map.entrySet()) {
                             AttributeModifier attributemodifier = (AttributeModifier)entry.getValue();
@@ -246,7 +246,7 @@
                     }
                 }
             } else {
-                Object object = StatCollector.func_74838_a("potion.empty").trim();
+                String object = StatCollector.func_74838_a("potion.empty").trim();
                 p_77624_3_.add(EnumChatFormatting.GRAY + object);
             }
 
