--- a/net/minecraft/item/Item.java
+++ b/net/minecraft/item/Item.java
@@ -486,7 +486,7 @@
             Blocks.field_150466_ao
         );
 
-        for (String s : Block.field_149771_c.func_148742_b()) {
+        for (String s : (Iterable<String>)Block.field_149771_c.func_148742_b()) {
             Block block = (Block)Block.field_149771_c.func_82594_a(s);
             Object object;
             if (block == Blocks.field_150325_L) {
