--- a/net/minecraft/network/status/server/S01PacketPong.java
+++ b/net/minecraft/network/status/server/S01PacketPong.java
@@ -7,7 +7,7 @@
 import net.neoforged.api.distmarker.Dist;
 import net.neoforged.api.distmarker.OnlyIn;
 
-public class S01PacketPong extends Packet {
+public class S01PacketPong extends Packet<INetHandlerStatusClient> {
     private long field_149293_a;
 
     public S01PacketPong() {
