--- a/net/minecraft/entity/passive/EntityVillager.java
+++ b/net/minecraft/entity/passive/EntityVillager.java
@@ -493,7 +493,7 @@
     }
 
     private static int func_146092_b(Item p_146092_0_, Random p_146092_1_) {
-        Tuple tuple = (Tuple)field_70958_bB.get(p_146092_0_);
+        Tuple<Integer, Integer> tuple = (Tuple)field_70958_bB.get(p_146092_0_);
         if (tuple == null) {
             return 1;
         } else {
@@ -521,7 +521,7 @@
     }
 
     private static int func_146090_c(Item p_146090_0_, Random p_146090_1_) {
-        Tuple tuple = (Tuple)field_70960_bC.get(p_146090_0_);
+        Tuple<Integer, Integer> tuple = (Tuple)field_70960_bC.get(p_146090_0_);
         if (tuple == null) {
             return 1;
         } else {
