--- a/net/minecraft/enchantment/EnchantmentHelper.java
+++ b/net/minecraft/enchantment/EnchantmentHelper.java
@@ -231,7 +231,7 @@
     }
 
     public static ItemStack func_77504_a(Random p_77504_0_, ItemStack p_77504_1_, int p_77504_2_) {
-        List list = func_77513_b(p_77504_0_, p_77504_1_, p_77504_2_);
+        List<EnchantmentData> list = func_77513_b(p_77504_0_, p_77504_1_, p_77504_2_);
         boolean flag = p_77504_1_.func_77973_b() == Items.field_151122_aG;
         if (flag) {
             p_77504_1_.func_150996_a(Items.field_151134_bR);
@@ -265,7 +265,7 @@
                 k = 1;
             }
 
-            ArrayList arraylist = null;
+            ArrayList<EnchantmentData> arraylist = null;
             Map map = func_77505_b(k, p_77513_1_);
             if (map != null && !map.isEmpty()) {
                 EnchantmentData enchantmentdata = (EnchantmentData)WeightedRandom.func_76271_a(p_77513_0_, map.values());
