--- a/net/minecraft/network/play/client/C02PacketUseEntity.java
+++ b/net/minecraft/network/play/client/C02PacketUseEntity.java
@@ -7,7 +7,7 @@
 import net.minecraft.network.play.INetHandlerPlayServer;
 import net.minecraft.world.World;
 
-public class C02PacketUseEntity extends Packet {
+public class C02PacketUseEntity extends Packet<INetHandlerPlayServer> {
     private int field_149567_a;
     private C02PacketUseEntity.Action field_149566_b;
 
