--- a/net/minecraft/network/play/client/C08PacketPlayerBlockPlacement.java
+++ b/net/minecraft/network/play/client/C08PacketPlayerBlockPlacement.java
@@ -6,7 +6,7 @@
 import net.minecraft.network.PacketBuffer;
 import net.minecraft.network.play.INetHandlerPlayServer;
 
-public class C08PacketPlayerBlockPlacement extends Packet {
+public class C08PacketPlayerBlockPlacement extends Packet<INetHandlerPlayServer> {
     private int field_149583_a;
     private int field_149581_b;
     private int field_149582_c;
