--- a/net/minecraft/network/play/client/C0CPacketInput.java
+++ b/net/minecraft/network/play/client/C0CPacketInput.java
@@ -5,7 +5,7 @@
 import net.minecraft.network.PacketBuffer;
 import net.minecraft.network.play.INetHandlerPlayServer;
 
-public class C0CPacketInput extends Packet {
+public class C0CPacketInput extends Packet<INetHandlerPlayServer> {
     private float field_149624_a;
     private float field_149622_b;
     private boolean field_149623_c;
