--- a/net/minecraft/network/play/client/C11PacketEnchantItem.java
+++ b/net/minecraft/network/play/client/C11PacketEnchantItem.java
@@ -5,7 +5,7 @@
 import net.minecraft.network.PacketBuffer;
 import net.minecraft.network.play.INetHandlerPlayServer;
 
-public class C11PacketEnchantItem extends Packet {
+public class C11PacketEnchantItem extends Packet<INetHandlerPlayServer> {
     private int field_149541_a;
     private int field_149540_b;
 
