--- a/net/minecraft/server/management/PreYggdrasilConverter.java
+++ b/net/minecraft/server/management/PreYggdrasilConverter.java
@@ -42,7 +42,7 @@
 
     @OnlyIn(Dist.DEDICATED_SERVER)
     static List<String> func_152721_a(File p_152721_0_, Map<String, String[]> p_152721_1_) throws IOException {
-        List list = Files.readLines(p_152721_0_, Charsets.UTF_8);
+        List<String> list = Files.readLines(p_152721_0_, Charsets.UTF_8);
 
         for (String s : list) {
             s = s.trim();
@@ -144,7 +144,7 @@
             }
 
             try {
-                HashMap hashmap = Maps.newHashMap();
+                HashMap<String, String[]> hashmap = Maps.newHashMap();
                 func_152721_a(field_152728_a, hashmap);
 
                 for (String s : hashmap.keySet()) {
@@ -302,7 +302,7 @@
         final File file3 = new File(file1.getParentFile(), "unknownplayers");
         if (file1.exists() && file1.isDirectory()) {
             File[] afile = file1.listFiles();
-            ArrayList arraylist = Lists.newArrayList();
+            ArrayList<String> arraylist = Lists.newArrayList();
 
             for (File file4 : afile) {
                 String s = file4.getName();
