--- a/net/minecraft/server/management/UserList.java
+++ b/net/minecraft/server/management/UserList.java
@@ -125,7 +125,7 @@
         }
 
         for (Object object : arraylist) {
-            this.field_152696_d.remove(object);
+            this.field_152696_d.remove(func_152681_a((U)object));
         }
     }
 
@@ -152,7 +152,7 @@
 
     @OnlyIn(Dist.DEDICATED_SERVER)
     public void func_152679_g() throws IOException {
-        Object object = null;
+        Collection<T> object = null;
         BufferedReader bufferedreader = null;
 
         try {
@@ -165,7 +165,7 @@
         if (object != null) {
             this.field_152696_d.clear();
 
-            for (UserListEntry userlistentry : object) {
+            for (T userlistentry : object) {
                 if (userlistentry.func_152640_f() != null) {
                     this.field_152696_d.put(this.func_152681_a((U)userlistentry.func_152640_f()), (T)userlistentry);
                 }
@@ -177,13 +177,13 @@
         private Serializer() {
         }
 
-        public JsonElement func_152751_a(UserListEntry<U> p_152751_1_, Type p_152751_2_, JsonSerializationContext p_152751_3_) {
+        public JsonElement serialize(UserListEntry<U> p_152751_1_, Type p_152751_2_, JsonSerializationContext p_152751_3_) {
             JsonObject jsonobject = new JsonObject();
             p_152751_1_.func_152641_a(jsonobject);
             return jsonobject;
         }
 
-        public UserListEntry<U> func_152750_a(JsonElement p_152750_1_, Type p_152750_2_, JsonDeserializationContext p_152750_3_) {
+        public UserListEntry<U> deserialize(JsonElement p_152750_1_, Type p_152750_2_, JsonDeserializationContext p_152750_3_) {
             if (p_152750_1_.isJsonObject()) {
                 JsonObject jsonobject = p_152750_1_.getAsJsonObject();
                 return UserList.this.func_152682_a(jsonobject);
