--- a/net/minecraft/scoreboard/Score.java
+++ b/net/minecraft/scoreboard/Score.java
@@ -4,7 +4,7 @@
 import java.util.List;
 
 public class Score {
-    public static final Comparator field_96658_a = new Comparator() {
+    public static final Comparator field_96658_a = new Comparator<Score>() {
         public int compare(Score p_compare_1_, Score p_compare_2_) {
             if (p_compare_1_.func_96652_c() > p_compare_2_.func_96652_c()) {
                 return 1;
