--- a/net/minecraft/scoreboard/ScorePlayerTeam.java
+++ b/net/minecraft/scoreboard/ScorePlayerTeam.java
@@ -9,7 +9,7 @@
 public class ScorePlayerTeam extends Team {
     private final Scoreboard field_96677_a;
     private final String field_96675_b;
-    private final Set field_96676_c = new HashSet();
+    private final Set<String> field_96676_c = new HashSet();
     private String field_96673_d;
     private String field_96674_e = "";
     private String field_96671_f = "";
@@ -40,7 +40,7 @@
         }
     }
 
-    public Collection func_96670_d() {
+    public Collection<String> func_96670_d() {
         return this.field_96676_c;
     }
 
