--- a/net/minecraft/scoreboard/Scoreboard.java
+++ b/net/minecraft/scoreboard/Scoreboard.java
@@ -25,7 +25,7 @@
             throw new IllegalArgumentException("An objective with the name '" + p_96535_1_ + "' already exists!");
         } else {
             scoreobjective = new ScoreObjective(this, p_96535_1_, p_96535_2_);
-            Object object = this.field_96543_b.get(p_96535_2_);
+            List<ScoreObjective> object = this.field_96543_b.get(p_96535_2_);
             if (object == null) {
                 object = new ArrayList();
                 this.field_96543_b.put(p_96535_2_, (List<ScoreObjective>)object);
@@ -44,7 +44,7 @@
     }
 
     public Score func_96529_a(String p_96529_1_, ScoreObjective p_96529_2_) {
-        Object object = this.field_96544_c.get(p_96529_1_);
+        Map<ScoreObjective, Score> object = this.field_96544_c.get(p_96529_1_);
         if (object == null) {
             object = new HashMap();
             this.field_96544_c.put(p_96529_1_, (Map<ScoreObjective, Score>)object);
@@ -89,7 +89,7 @@
     }
 
     public Collection<Score> func_96528_e() {
-        Collection collection = this.field_96544_c.values();
+        Collection<Map<ScoreObjective, Score>> collection = this.field_96544_c.values();
         ArrayList arraylist = new ArrayList();
 
         for (Map map : collection) {
