--- a/net/minecraft/client/Minecraft.java
+++ b/net/minecraft/client/Minecraft.java
@@ -555,7 +555,7 @@
     }
 
     private void func_110441_Q() throws LWJGLException {
-        HashSet hashset = new HashSet();
+        HashSet<DisplayMode> hashset = new HashSet();
         Collections.addAll(hashset, Display.getAvailableDisplayModes());
         DisplayMode displaymode = Display.getDesktopDisplayMode();
         if (!hashset.contains(displaymode) && Util.func_110647_a() == Util.EnumOS.OSX) {
