--- a/net/minecraft/client/shader/ShaderManager.java
+++ b/net/minecraft/client/shader/ShaderManager.java
@@ -35,7 +35,7 @@
     private final Map field_147997_f = Maps.newHashMap();
     private final List field_147998_g = Lists.newArrayList();
     private final List field_148010_h = Lists.newArrayList();
-    private final List field_148011_i = Lists.newArrayList();
+    private final List<ShaderUniform> field_148011_i = Lists.newArrayList();
     private final List field_148008_j = Lists.newArrayList();
     private final Map field_148009_k = Maps.newHashMap();
     private final int field_148006_l;
@@ -44,7 +44,7 @@
     private boolean field_148005_o;
     private final JsonBlendingMode field_148016_p;
     private final List field_148015_q;
-    private final List field_148014_r;
+    private final List<String> field_148014_r;
     private final ShaderLoader field_148013_s;
     private final ShaderLoader field_148012_t;
 
