--- a/net/minecraft/client/gui/GuiChat.java
+++ b/net/minecraft/client/gui/GuiChat.java
@@ -44,7 +44,7 @@
     private boolean field_146417_i;
     private boolean field_146414_r;
     private int field_146413_s;
-    private List field_146412_t = new ArrayList();
+    private List<String> field_146412_t = new ArrayList();
     private URI field_146411_u;
     protected GuiTextField field_146415_a;
     private String field_146409_v = "";
