--- a/net/minecraft/client/gui/GuiFlatPresets.java
+++ b/net/minecraft/client/gui/GuiFlatPresets.java
@@ -116,7 +116,7 @@
         func_146421_a(p_146425_0_, p_146425_1_, p_146425_2_, null, p_146425_3_);
     }
 
-    private static void func_146421_a(String p_146421_0_, Item p_146421_1_, BiomeGenBase p_146421_2_, List p_146421_3_, FlatLayerInfo... p_146421_4_) {
+    private static void func_146421_a(String p_146421_0_, Item p_146421_1_, BiomeGenBase p_146421_2_, List<String> p_146421_3_, FlatLayerInfo... p_146421_4_) {
         FlatGeneratorInfo flatgeneratorinfo = new FlatGeneratorInfo();
 
         for (int i = p_146421_4_.length - 1; i >= 0; i--) {
