--- a/net/minecraft/client/gui/GuiSnooper.java
+++ b/net/minecraft/client/gui/GuiSnooper.java
@@ -29,7 +29,7 @@
     public void func_73866_w_() {
         this.field_146610_i = I18n.func_135052_a("options.snooper.title");
         String s = I18n.func_135052_a("options.snooper.desc");
-        ArrayList arraylist = new ArrayList();
+        ArrayList<String> arraylist = new ArrayList();
 
         for (String s1 : this.field_146289_q.func_78271_c(s, this.field_146294_l - 30)) {
             arraylist.add(s1);
@@ -47,13 +47,13 @@
         this.field_146292_n.add(new GuiButton(2, this.field_146294_l / 2 + 2, this.field_146295_m - 30, 150, 20, I18n.func_135052_a("gui.done")));
         boolean flag = this.field_146297_k.func_71401_C() != null && this.field_146297_k.func_71401_C().func_80003_ah() != null;
 
-        for (Entry entry : new TreeMap(this.field_146297_k.func_71378_E().func_76465_c()).entrySet()) {
+        for (Entry entry : new TreeMap<>(this.field_146297_k.func_71378_E().func_76465_c()).entrySet()) {
             this.field_146604_g.add((flag ? "C " : "") + (String)entry.getKey());
             this.field_146609_h.add(this.field_146289_q.func_78269_a((String)entry.getValue(), this.field_146294_l - 220));
         }
 
         if (flag) {
-            for (Entry entry1 : new TreeMap(this.field_146297_k.func_71401_C().func_80003_ah().func_76465_c()).entrySet()) {
+            for (Entry entry1 : new TreeMap<>(this.field_146297_k.func_71401_C().func_80003_ah().func_76465_c()).entrySet()) {
                 this.field_146604_g.add("S " + (String)entry1.getKey());
                 this.field_146609_h.add(this.field_146289_q.func_78269_a((String)entry1.getValue(), this.field_146294_l - 220));
             }
