--- a/net/minecraft/client/gui/ServerSelectionList.java
+++ b/net/minecraft/client/gui/ServerSelectionList.java
@@ -59,7 +59,7 @@
         }
     }
 
-    public void func_148194_a(List p_148194_1_) {
+    public void func_148194_a(List<LanServerDetector.LanServer> p_148194_1_) {
         this.field_148199_m.clear();
 
         for (LanServerDetector.LanServer lanserverdetector$lanserver : p_148194_1_) {
