--- a/net/minecraft/world/storage/MapStorage.java
+++ b/net/minecraft/world/storage/MapStorage.java
@@ -17,9 +17,9 @@
 
 public class MapStorage {
     private ISaveHandler field_75751_a;
-    private Map field_75749_b = new HashMap();
-    private List field_75750_c = new ArrayList();
-    private Map field_75748_d = new HashMap();
+    private Map<String, WorldSavedData> field_75749_b = new HashMap();
+    private List<WorldSavedData> field_75750_c = new ArrayList();
+    private Map<String, Short> field_75748_d = new HashMap();
 
     public MapStorage(ISaveHandler p_i2162_1_) {
         this.field_75751_a = p_i2162_1_;
