--- a/net/minecraft/village/VillageCollection.java
+++ b/net/minecraft/village/VillageCollection.java
@@ -13,9 +13,9 @@
 
 public class VillageCollection extends WorldSavedData {
     private World field_75556_a;
-    private final List field_75554_b = new ArrayList();
-    private final List field_75555_c = new ArrayList();
-    private final List field_75552_d = new ArrayList();
+    private final List<ChunkCoordinates> field_75554_b = new ArrayList();
+    private final List<VillageDoorInfo> field_75555_c = new ArrayList();
+    private final List<Village> field_75552_d = new ArrayList();
     private int field_75553_e;
 
     public VillageCollection(String p_i1677_1_) {
@@ -71,7 +71,7 @@
         }
     }
 
-    public List func_75540_b() {
+    public List<Village> func_75540_b() {
         return this.field_75552_d;
     }
 
