--- a/net/minecraft/entity/EntityLivingBase.java
+++ b/net/minecraft/entity/EntityLivingBase.java
@@ -52,7 +52,7 @@
     private static final AttributeModifier field_110157_c = new AttributeModifier(field_110156_b, "Sprinting speed boost", 0.3F, 2).func_111168_a(false);
     private BaseAttributeMap field_110155_d;
     private final CombatTracker field_94063_bt = new CombatTracker(this);
-    private final HashMap field_70713_bf = new HashMap();
+    private final HashMap<Integer, PotionEffect> field_70713_bf = new HashMap();
     private final ItemStack[] field_82180_bT = new ItemStack[5];
     public boolean field_82175_bq;
     public int field_110158_av;
@@ -508,7 +508,7 @@
         }
     }
 
-    public Collection func_70651_bq() {
+    public Collection<PotionEffect> func_70651_bq() {
         return this.field_70713_bf.values();
     }
 
