--- a/net/minecraft/client/resources/DefaultResourcePack.java
+++ b/net/minecraft/client/resources/DefaultResourcePack.java
@@ -15,7 +15,7 @@
 import net.minecraft.util.ResourceLocation;
 
 public class DefaultResourcePack implements IResourcePack {
-    public static final Set field_110608_a = ImmutableSet.of("minecraft", "realms");
+    public static final Set<String> field_110608_a = ImmutableSet.of("minecraft", "realms");
     private final Map field_152781_b;
 
     public DefaultResourcePack(Map p_i46346_1_) {
@@ -52,7 +52,7 @@
     }
 
     @Override
-    public Set func_110587_b() {
+    public Set<String> func_110587_b() {
         return field_110608_a;
     }
 
