--- a/net/minecraft/client/resources/IReloadableResourceManager.java
+++ b/net/minecraft/client/resources/IReloadableResourceManager.java
@@ -3,7 +3,7 @@
 import java.util.List;
 
 public interface IReloadableResourceManager extends IResourceManager {
-    void func_110541_a(List var1);
+    void func_110541_a(List<IResourcePack> var1);
 
     void func_110542_a(IResourceManagerReloadListener var1);
 }
