--- a/net/minecraft/client/resources/IResourceManager.java
+++ b/net/minecraft/client/resources/IResourceManager.java
@@ -6,9 +6,9 @@
 import net.minecraft.util.ResourceLocation;
 
 public interface IResourceManager {
-    Set func_135055_a();
+    Set<String> func_135055_a();
 
     IResource func_110536_a(ResourceLocation var1) throws IOException;
 
-    List func_135056_b(ResourceLocation var1) throws IOException;
+    List<IResource> func_135056_b(ResourceLocation var1) throws IOException;
 }
