--- a/net/minecraft/client/resources/Locale.java
+++ b/net/minecraft/client/resources/Locale.java
@@ -16,10 +16,10 @@
 public class Locale {
     private static final Splitter field_135030_b = Splitter.on('=').limit(2);
     private static final Pattern field_135031_c = Pattern.compile("%(\\d+\\$)?[\\d\\.]*[df]");
-    Map field_135032_a = Maps.newHashMap();
+    Map<String, String> field_135032_a = Maps.newHashMap();
     private boolean field_135029_d;
 
-    public synchronized void func_135022_a(IResourceManager p_135022_1_, List p_135022_2_) {
+    public synchronized void func_135022_a(IResourceManager p_135022_1_, List<String> p_135022_2_) {
         this.field_135032_a.clear();
 
         for (String s : p_135022_2_) {
@@ -60,7 +60,7 @@
         this.field_135029_d = f > 0.1;
     }
 
-    private void func_135028_a(List p_135028_1_) throws IOException {
+    private void func_135028_a(List<IResource> p_135028_1_) throws IOException {
         for (IResource iresource : p_135028_1_) {
             this.func_135021_a(iresource.func_110527_b());
         }
