--- a/net/minecraft/client/resources/ResourcePackRepository.java
+++ b/net/minecraft/client/resources/ResourcePackRepository.java
@@ -132,7 +132,7 @@
     }
 
     private void func_148528_a(String p_148528_1_, File p_148528_2_) {
-        HashMap hashmap = Maps.newHashMap();
+        HashMap<String, String> hashmap = Maps.newHashMap();
         GuiScreenWorking guiscreenworking = new GuiScreenWorking();
         hashmap.put("X-Minecraft-Username", Minecraft.func_71410_x().func_110432_I().func_111285_a());
         hashmap.put("X-Minecraft-UUID", Minecraft.func_71410_x().func_110432_I().func_148255_b());
