--- a/net/minecraft/client/renderer/tileentity/RenderEndPortal.java
+++ b/net/minecraft/client/renderer/tileentity/RenderEndPortal.java
@@ -11,7 +11,7 @@
 import net.minecraft.util.ResourceLocation;
 import org.lwjgl.opengl.GL11;
 
-public class RenderEndPortal extends TileEntitySpecialRenderer {
+public class RenderEndPortal extends TileEntitySpecialRenderer<TileEntityEndPortal> {
     private static final ResourceLocation field_147529_c = new ResourceLocation("textures/environment/end_sky.png");
     private static final ResourceLocation field_147526_d = new ResourceLocation("textures/entity/end_portal.png");
     private static final Random field_147527_e = new Random(31100L);
