--- a/net/minecraft/client/renderer/tileentity/TileEntitySignRenderer.java
+++ b/net/minecraft/client/renderer/tileentity/TileEntitySignRenderer.java
@@ -8,7 +8,7 @@
 import net.minecraft.util.ResourceLocation;
 import org.lwjgl.opengl.GL11;
 
-public class TileEntitySignRenderer extends TileEntitySpecialRenderer {
+public class TileEntitySignRenderer extends TileEntitySpecialRenderer<TileEntitySign> {
     private static final ResourceLocation field_147513_b = new ResourceLocation("textures/entity/sign.png");
     private final ModelSign field_147514_c = new ModelSign();
 
