--- a/net/minecraft/client/renderer/tileentity/TileEntityRendererPiston.java
+++ b/net/minecraft/client/renderer/tileentity/TileEntityRendererPiston.java
@@ -13,7 +13,7 @@
 import net.minecraft.world.World;
 import org.lwjgl.opengl.GL11;
 
-public class TileEntityRendererPiston extends TileEntitySpecialRenderer {
+public class TileEntityRendererPiston extends TileEntitySpecialRenderer<TileEntityPiston> {
     private RenderBlocks field_147516_b;
 
     public void func_147500_a(TileEntityPiston p_147500_1_, double p_147500_2_, double p_147500_4_, double p_147500_6_, float p_147500_8_) {
