--- a/net/minecraft/network/status/client/C01PacketPing.java
+++ b/net/minecraft/network/status/client/C01PacketPing.java
@@ -7,7 +7,7 @@
 import net.neoforged.api.distmarker.Dist;
 import net.neoforged.api.distmarker.OnlyIn;
 
-public class C01PacketPing extends Packet {
+public class C01PacketPing extends Packet<INetHandlerStatusServer> {
     private long field_149290_a;
 
     public C01PacketPing() {
