--- a/net/minecraft/entity/item/EntityItem.java
+++ b/net/minecraft/entity/item/EntityItem.java
@@ -125,7 +125,7 @@
     }
 
     private void func_85054_d() {
-        for (EntityItem entityitem : this.field_70170_p.func_72872_a(EntityItem.class, this.field_70121_D.func_72314_b(0.5, 0.0, 0.5))) {
+        for (EntityItem entityitem : (Iterable<EntityItem>)this.field_70170_p.func_72872_a(EntityItem.class, this.field_70121_D.func_72314_b(0.5, 0.0, 0.5))) {
             this.func_70289_a(entityitem);
         }
     }
