--- a/net/minecraft/realms/DisconnectedOnlineScreen.java
+++ b/net/minecraft/realms/DisconnectedOnlineScreen.java
@@ -9,7 +9,7 @@
 public class DisconnectedOnlineScreen extends RealmsScreen {
     private String title;
     private IChatComponent reason;
-    private List lines;
+    private List<String> lines;
     private final RealmsScreen parent;
 
     public DisconnectedOnlineScreen(RealmsScreen p_i1000_1_, String p_i1000_2_, IChatComponent p_i1000_3_) {
