--- a/net/minecraft/realms/RealmsLevelSummary.java
+++ b/net/minecraft/realms/RealmsLevelSummary.java
@@ -5,7 +5,7 @@
 import net.neoforged.api.distmarker.OnlyIn;
 
 @OnlyIn(Dist.CLIENT)
-public class RealmsLevelSummary implements Comparable {
+public class RealmsLevelSummary implements Comparable<RealmsLevelSummary> {
     private SaveFormatComparator levelSummary;
 
     public RealmsLevelSummary(SaveFormatComparator p_i1109_1_) {
