--- a/net/minecraft/client/renderer/texture/Stitcher.java
+++ b/net/minecraft/client/renderer/texture/Stitcher.java
@@ -71,7 +71,7 @@
     }
 
     public List<TextureAtlasSprite> func_94309_g() {
-        ArrayList arraylist = Lists.newArrayList();
+        ArrayList<Stitcher.Slot> arraylist = Lists.newArrayList();
 
         for (Stitcher.Slot stitcher$slot : this.field_94317_b) {
             stitcher$slot.func_94184_a(arraylist);
