--- a/net/minecraft/network/status/client/C00PacketServerQuery.java
+++ b/net/minecraft/network/status/client/C00PacketServerQuery.java
@@ -5,7 +5,7 @@
 import net.minecraft.network.PacketBuffer;
 import net.minecraft.network.status.INetHandlerStatusServer;
 
-public class C00PacketServerQuery extends Packet {
+public class C00PacketServerQuery extends Packet<INetHandlerStatusServer> {
     @Override
     public void func_148837_a(PacketBuffer p_148837_1_) throws IOException {
     }
